<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Rachel Ancar" />

<meta name="date" content="2018-11-25" />

<title>Analysis of rRNA to Detect EndoU Cleavage Sites from experiemnt 2</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}

.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">EndoU Collaboration Data Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="MHV_endoU_analysis_final.html">MHV RNA analysis exp.1</a>
</li>
<li>
  <a href="MHV_endoU_analysis_experiment2.html">MHV RNA analysis exp.2</a>
</li>
<li>
  <a href="Alignments.html">Alignments</a>
</li>
<li>
  <a href="rRNA_endoU_analysis_final.html">rRNA analysis exp.1</a>
</li>
<li>
  <a href="rRNA_endoU_analysis_exp2_final.html">rRNA analysis exp.2</a>
</li>
<li>
  <a href="mRNA_endoU_analysis.html">mRNA analysis exp.1</a>
</li>
<li>
  <a href="mRNA_endoU_exp2_analysis.html">mRNA analysis exp.2</a>
</li>
<li>
  <a href="U6_23-Cyclic_Phosphate_Analysis.html">U6 RNA analysis</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">

<div class="btn-group pull-right">
<button type="button" class="btn btn-default btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Analysis of rRNA to Detect EndoU Cleavage Sites from experiemnt 2</h1>
<h4 class="author"><em>Rachel Ancar</em></h4>
<h4 class="date"><em>11/25/2018</em></h4>

</div>


<p>Prior to this analysis, 2´3´-cyclic phosphate RNA-sequencing libraries were prepared from mouse macrophages: B6 (WT), IFNAR KO, RNaseL KO. These cells were either mock infected or infected with WT MHV virus (from Volker or Susan), MHV mutated to inactivate ns2 (phosphodiesterase) activity, or MHV mutated to inactivate nsp15 (endoribonuclease) activity for 9 and 12 hours.</p>
<p>These libraries were processed to generate bedgraph files containing sites of 3´-cleavage and the associated dinucleotide for 18S, 28S, 5.8s and 5s rRNA.</p>
<pre class="r"><code>library(tidyverse)
library(readr)
library(dplyr)
library(ggplot2)
library(stringr)
library(cowplot)
library(RColorBrewer)
library(purrr)
library(shiny)
library(DT)</code></pre>
<div id="coverage-plots-for-18s-28s-and-5s-rrna-by-cell-type-and-type-of-viral-infection" class="section level1">
<h1>Coverage plots for 18S, 28S, and 5s rRNA by cell type and type of viral infection</h1>
<p>These plots display coverage across the entire 18S, 28S, and 5s rRNA sequence. Counts are normalized by library size.</p>
<div id="s-rrna" class="section level2">
<h2>18s rRNA</h2>
<pre class="r"><code>sample_info_table &lt;- read_tsv(&quot;sample_info.txt&quot;, col_names = c(&quot;cell type&quot;, &quot;virus type&quot;, &quot;time&quot;))
datatable(sample_info_table)</code></pre>
<div id="htmlwidget-55c054a073b6129866cf" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-55c054a073b6129866cf">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36"],["B6","B6","B6","B6","B6","IFNAR","IFNAR","IFNAR","IFNAR","IFNAR","RnaseL","RnaseL","RnaseL","RnaseL","RnaseL","B6","B6","B6","B6","B6","IFNAR","IFNAR","IFNAR","IFNAR","IFNAR","RnaseL","RnaseL","RnaseL","RnaseL","RnaseL","B6","IFNAR","RNaseL","B6","IFNAR","RNaseL"],["Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","mock","mock","mock","mock","mock","mock"],["12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","9","9","9","9","9","9","9","9","9","9","9","9","9","9","9","0/12","0/12","0/12","0/9","0/9","0/9"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>cell type<\/th>\n      <th>virus type<\/th>\n      <th>time<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}]}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>#18S rRNA coverage plots

data_dir_18S = &quot;~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/18S/exp1/&quot;
data_files_18S = list.files(data_dir_18S, full.names = T)

read_file &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df$name &lt;- basename(x)
  df
}

table_18S &lt;- purrr::map_df(data_files_18S, read_file) %&gt;%
  mutate(name = str_replace(name, &quot;.r18s.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;), sep = &#39;_&#39;)

plot_18S &lt;- ggplot(table_18S, aes(x = start, y = normalized_count, fill = time, width = 5)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette=&quot;Set1&quot;) +
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) + 
    scale_x_continuous(breaks=seq(0, 2000, 250)) + 
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    ggtitle(&quot;18S rRNA coverage plots&quot;)
plot_18S</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-2-2.png" width="672" /></p>
<pre class="r"><code># Previously documented RNaseL cleavage site 542 and 771 in 18S rRNA

plot_18S_542 &lt;- ggplot(table_18S, aes(x = start, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    xlim(530, 560) +
    ggtitle(&quot;RNaseL cleavage site 542 in 18S rRNA&quot;)
plot_18S_542</code></pre>
<pre><code>## Warning: Removed 58140 rows containing missing values (geom_bar).</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-2-3.png" width="672" /></p>
<pre class="r"><code>plot_18S_771 &lt;- ggplot(table_18S, aes(x = start, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    xlim(760, 780) +
    ggtitle(&quot;RNaseL cleavage site 771 in 18S rRNA&quot;)
plot_18S_771</code></pre>
<pre><code>## Warning: Removed 58481 rows containing missing values (geom_bar).</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-2-4.png" width="672" /></p>
</div>
<div id="s-rrna-1" class="section level2">
<h2>28s rRNA</h2>
<pre class="r"><code>#28S rRNA coverage plots

data_files_28S = list.files(&quot;~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/28S/exp1/&quot; , full.names = T)

read_file &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df$name &lt;- basename(x)
  df
}

table_28S &lt;- purrr::map_df(data_files_28S, read_file) %&gt;%
  mutate(name = str_replace(name, &quot;.r28s.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;), sep = &#39;_&#39;)

plot_28S &lt;- ggplot(table_28S, aes(x = end, y = normalized_count, fill = time, width = 10)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette=&quot;Set1&quot;) +
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
    scale_x_continuous(breaks=seq(0, 5000, 500)) + 
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    ggtitle(&quot;28S rRNA coverage plots&quot;)
plot_28S</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-3-1.png" width="672" /></p>
</div>
</div>
<div id="s-rrna-2" class="section level1">
<h1>5S rRNA</h1>
<pre class="r"><code>data_dir_5S = &quot;~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/5S/&quot;
data_files_5S = list.files(data_dir_5S, full.names = T)

read_file &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df$name &lt;- basename(x)
  df
}

table_5S &lt;- purrr::map_df(data_files_5S, read_file) %&gt;%
  mutate(name = str_replace(name, &quot;.r5s.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;), sep = &#39;_&#39;)

plot_5S &lt;- ggplot(table_5S, aes(x = end, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette=&quot;Set1&quot;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
    scale_x_continuous(breaks=seq(0, 200, 20)) + 
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    ggtitle(&quot;5S rRNA coverage plots&quot;)
plot_5S</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-4-1.png" width="672" /></p>
<div id="s-rrna-3" class="section level2">
<h2>5.8s rRNA</h2>
<pre class="r"><code>data_dir_5.8S = &quot;~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/5.8S/exp1/&quot;
data_files_5.8S = list.files(data_dir_5.8S, full.names = T)

read_file &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df$name &lt;- basename(x)
  df
}

table_5.8S &lt;- purrr::map_df(data_files_5.8S, read_file) %&gt;%
  mutate(name = str_replace(name, &quot;.r5.8s.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;), sep = &#39;_&#39;)

plot_5.8S &lt;- ggplot(table_5.8S, aes(x = end, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette=&quot;Set1&quot;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
    scale_x_continuous(breaks=seq(0, 200, 20)) + 
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    ggtitle(&quot;5.8S rRNA coverage plots&quot;)
plot_5.8S</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-5-1.png" width="672" /></p>
</div>
<div id="disscusion-of-ribosomal-rna-coverage-plots" class="section level2">
<h2>Disscusion of ribosomal RNA coverage plots</h2>
<p>First, it is clear that there is signfiicant non-specific cleavage of ribosomal RNA that has been captured in these libaries. There is robust cleavage in the mock samples of all analyzed rRNAs. Across all rRNAs and conditions, most cleavage is detected at 9 vs. 12 hpi. The exception to this observation is for rRNA cleavage from B6 cells infected with MHV ns2 mutant. In these cells, there is more clevage at 12 hpi, which may be a reflection of specific increased RNase L activation. The 5s and 5.8s rRNA signal does not follow these patterns observed in the 18s and 28 rRNA. The signal in general is much lower and there do not seem to be any speciifc changes in signal captured across cell type or infection.</p>
<p>Speciifc cleavage can still be detected at the known RNase L cleavage site in 18s rRNA at nucleotide 542 and 771. At this site, there is very little cleavage in the mock samples, and clear induction of cleavage at this site in B6 cells infected with the nsp15 mutant relative to the other virus types.</p>
</div>
</div>
<div id="rnase-l-substractive-coverage-analysis-of-rrna-reads-for-all-cell-types-by-type-of-viral-infection" class="section level1">
<h1>RNase L substractive coverage analysis of rRNA reads for all cell types by type of viral infection</h1>
<p>The normalized counts detected in RNase L KO cells for each type of viral infection were subtracted from the normalized counts detected in WT B6 and IFNAR KO cells. This substractive analysis will emphasize RNase L dependent cleavage events by reporting signals that occur only in the presence of RNase L.</p>
<pre class="r"><code>#18S rRNA RNase L substractive analysis 

subtractive_18S &lt;- table_18S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count) 
subtractive_18S[is.na(subtractive_18S)] &lt;- 0

subtractive_18S &lt;- subtractive_18S %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;%                     
  dplyr::select(start:end, virus:time, B6_RNaseL:IFNAR_RNaseL) %&gt;% 
  gather(cell, normalized_count, B6_RNaseL:IFNAR_RNaseL)
                                      
#18s rRNA RNase L dependent cleavage plots

sub18S_plot &lt;- ggplot(subtractive_18S, aes(x = end, y = normalized_count, fill = time, width = 5)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    #geom_line(aes(color = time)) + 
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) + 
    scale_y_continuous(limits = c(0 , 0.3)) +
    scale_x_continuous(breaks=seq(0, 2000, 250)) + 
    ggtitle(&quot;18s rRNA RNase L dependent cleavage plots&quot;)
sub18S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-6-1.png" width="672" /></p>
<pre class="r"><code>#28S rRNA RNase L substractive analysis 

subtractive_28S &lt;- table_28S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count) 
subtractive_28S[is.na(subtractive_28S)] &lt;- 0

subtractive_28S &lt;- subtractive_28S %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(start:end, virus:time, B6_RNaseL:IFNAR_RNaseL) %&gt;%      
  gather(cell, normalized_count, B6_RNaseL:IFNAR_RNaseL)

#28s rRNA RNase L dependent cleavage plots
                                      
sub28S_plot &lt;- ggplot(subtractive_28S, aes(x = end, y = normalized_count, fill = time, width = 10)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +                  scale_y_continuous(limits = c(0 , 0.3)) +
    scale_x_continuous(breaks=seq(0, 5000, 500)) + 
   ggtitle(&quot;28s rRNA RNase L dependent cleavage plots&quot;)
sub28S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-6-2.png" width="672" /></p>
<pre class="r"><code>#5S rRNA RNase L substractive analysis 

subtractive_5S &lt;- table_5S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count) 
subtractive_5S[is.na(subtractive_5S)] &lt;- 0

subtractive_5S &lt;- subtractive_5S %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(start:end, virus:time, B6_RNaseL:IFNAR_RNaseL) %&gt;%    
  gather(cell, normalized_count, B6_RNaseL:IFNAR_RNaseL)

#5s rRNA RNase L dependent cleavage plots
                                      
sub5S_plot &lt;- ggplot(subtractive_5S, aes(x = end, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +              
    scale_y_continuous(limits = c(0 , 0.03)) +
    scale_x_continuous(breaks=seq(0, 200, 20)) + 
    ggtitle(&quot;5s rRNA RNase L dependent cleavage plots&quot;)
sub5S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-6-3.png" width="672" /></p>
<pre class="r"><code>#5.8S rRNA RNase L substractive analysis 

subtractive_5.8S &lt;- table_5.8S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count) 
subtractive_5.8S[is.na(subtractive_5.8S)] &lt;- 0

subtractive_5.8S &lt;- subtractive_5.8S %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(start:end, virus:time, B6_RNaseL:IFNAR_RNaseL) %&gt;%    
  gather(cell, normalized_count, B6_RNaseL:IFNAR_RNaseL)

#5s rRNA RNase L dependent cleavage plots
                                      
sub5.8S_plot &lt;- ggplot(subtractive_5.8S, aes(x = end, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +              
    scale_y_continuous(limits = c(0 , 0.03)) +
    scale_x_continuous(breaks=seq(0, 200, 20)) + 
    ggtitle(&quot;5.8s rRNA RNase L dependent cleavage plots&quot;)
sub5.8S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-6-4.png" width="672" /></p>
</div>
<div id="endou-mutant-nsp15-substractive-coverage-analysis-of-mhv-strand-for-all-cell-types-by-type-of-viral-infection" class="section level1">
<h1>EndoU mutant (nsp15) substractive coverage analysis of MHV (+) strand for all cell types by type of viral infection</h1>
<p>This is very similar to the analysis above, expect it will substract signal occuring in the abscence of nsp15 actviity, which will emphasize sites specific to nsp15 cleavage actviity.</p>
<pre class="r"><code>#18S rRNA EndoU substractive analysis 

subtractive_nsp15_18S &lt;- table_18S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_18S[is.na(subtractive_nsp15_18S)] &lt;- 0

subtractive_nsp15_18S &lt;- subtractive_nsp15_18S %&gt;%       
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;%
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;%
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;%
  mutate(mock_nsp15 = mock - nsp15) %&gt;%
  dplyr::select(start:end, cell:time, MHVS_nsp15:mock_nsp15) %&gt;% 
  gather(virus, normalized_count, MHVS_nsp15:mock_nsp15)
                                      
subnsp15_18S_plot &lt;- ggplot(subtractive_nsp15_18S, aes(x = end, y = normalized_count, fill = time, width = 5)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) + 
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +  
    scale_y_continuous(limits = c(0.0, 0.2)) +
    scale_x_continuous(breaks=seq(0, 2000, 250)) + 
    ggtitle(&quot;18S rRNA EndoU substractive analysis &quot;)
subnsp15_18S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-7-1.png" width="672" /></p>
<pre class="r"><code>#28S

subtractive_nsp15_28S &lt;- table_28S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_28S[is.na(subtractive_nsp15_28S)] &lt;- 0

subtractive_nsp15_28S &lt;- subtractive_nsp15_28S %&gt;%   
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(start:end, cell:time, MHVS_nsp15:mock_nsp15) %&gt;%    
  gather(virus, normalized_count, MHVS_nsp15:mock_nsp15)
                                      
subnsp15_28S_plot &lt;- ggplot(subtractive_nsp15_28S, aes(x = end, y = normalized_count, fill = time, width = 10)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +     
    scale_y_continuous(limits = c(0.0, 0.2)) +
    scale_x_continuous(breaks=seq(0, 5000, 500)) + 
    ggtitle(&quot;28S rRNA EndoU substractive analysis &quot;)
subnsp15_28S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-7-2.png" width="672" /></p>
<pre class="r"><code>#5S

subtractive_nsp15_5S &lt;- table_5S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_5S[is.na(subtractive_nsp15_5S)] &lt;- 0

subtractive_nsp15_5S &lt;- subtractive_nsp15_5S %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(start:end, cell:time, MHVS_nsp15:mock_nsp15) %&gt;%
  gather(virus, normalized_count, MHVS_nsp15:mock_nsp15)
                                      
subnsp15_5S_plot &lt;- ggplot(subtractive_nsp15_5S, aes(x = end, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +                   scale_y_continuous(limits = c(0.0, 0.02)) +
    scale_x_continuous(breaks=seq(0, 200, 20)) + 
    ggtitle(&quot;5S rRNA EndoU substractive analysis &quot;)
subnsp15_5S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-7-3.png" width="672" /></p>
<pre class="r"><code>#5.8S

subtractive_nsp15_5.8S &lt;- table_5.8S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_5.8S[is.na(subtractive_nsp15_5.8S)] &lt;- 0

subtractive_nsp15_5.8S &lt;- subtractive_nsp15_5.8S %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(start:end, cell:time, MHVS_nsp15:mock_nsp15) %&gt;%
  gather(virus, normalized_count, MHVS_nsp15:mock_nsp15)
                                      
subnsp15_5.8S_plot &lt;- ggplot(subtractive_nsp15_5.8S, aes(x = end, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +                   scale_y_continuous(limits = c(0.0, 0.02)) +
    scale_x_continuous(breaks=seq(0, 200, 20)) + 
    ggtitle(&quot;5.8S rRNA EndoU substractive analysis &quot;)
subnsp15_5.8S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-7-4.png" width="672" /></p>
<div id="fold-change-for-rnasel-acitivity-by-rrna" class="section level2">
<h2>Fold-change for RNaseL acitivity by rRNA</h2>
<p>In this analysis, all positions in the MHV positive strand RNA with more than 20 reads were queried to identify sites with equal to or greater than a 5-fold decrease in signal comparing B6 WT cells and IFNAR KO cells to RNaseL KO cells (B6/RNaseL and IFNAR/RNaseL).</p>
<p>This analysis is independent of viral infection and just asks which sites change the most when RNaseL is absent.</p>
</div>
<div id="s" class="section level2">
<h2>18s</h2>
<pre class="r"><code>change_RNaseL_18S &lt;- table_18S %&gt;% 
  filter(count &gt;= 20) %&gt;%
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count)
change_RNaseL_18S[is.na(change_RNaseL_18S)] &lt;- 0

change_RNaseL_18S &lt;- change_RNaseL_18S %&gt;% 
  mutate(B6_change = B6/RNaseL) %&gt;% 
  mutate(IFNAR_change = IFNAR/RNaseL) %&gt;% 
  gather(cell_comp, fold_change, B6_change:IFNAR_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_RNaseL_18S)</code></pre>
<div id="htmlwidget-8fee3ba43cecf85f5753" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-8fee3ba43cecf85f5753">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51"],["r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|"],[541,541,770,542,541,541,1711,541,540,539,872,1714,1807,1724,527,1722,1811,1487,1487,1864,1723,139,1505,135,1695,241,532,1813,317,634,1732,1768,1676,1728,633,1730,228,1725,1058,134,1774,1804,308,1826,1827,1767,1711,135,1774,173,360],[542,542,771,543,542,542,1712,542,541,540,873,1715,1808,1725,528,1723,1812,1488,1488,1865,1724,140,1506,136,1696,242,533,1814,318,635,1733,1769,1677,1729,634,1731,229,1726,1059,135,1775,1805,309,1827,1828,1768,1712,136,1775,174,361],["nsp15","nsp15","nsp15","nsp15","ns2","ns2","nsp15","MHVV","nsp15","nsp15","nsp15","nsp15","MHVV","ns2","nsp15","ns2","MHVV","mock","mock","nsp15","ns2","MHVV","mock","nsp15","mock","nsp15","nsp15","MHVV","nsp15","ns2","ns2","ns2","mock","ns2","ns2","ns2","MHVV","ns2","nsp15","nsp15","MHVS","ns2","nsp15","MHVV","MHVV","nsp15","nsp15","nsp15","ns2","MHVS","nsp15"],["12","9","12","12","9","12","12","9","12","12","12","12","12","9","9","9","12","12","12","9","9","12","12","9","12","12","12","12","9","9","9","9","12","9","9","9","12","9","12","9","12","9","9","12","12","12","9","12","12","12","9"],[0.288548,0.221025,0.164093,0.159939,0.139303,0.0969028,0.167209,0.0722722,0.0941633,0.0446583,0.0721803,0.117531,0.00828085,0.261768,0.0807092,0.202067,0.0085244,0.339028,0.339028,0.0912606,0.154612,0.0199715,0.24559,0.112364,0.294834,0.0631794,0.0282144,0.0104728,0.135503,0.0535783,0.254114,0.071948,0.261374,0.148488,0.203597,0.13318,0,0.125526,0.127224,0.173821,0.0177795,0.047455,0.0283223,0.0151004,0.028983,0.0501973,0.093482,0.150073,0.00573766,0.0260542,0.0644193],[0.0162298,0.0154943,0.00414897,0.0198907,0,0.0134265,0.0183043,0.0116939,0.00585737,0.00353883,0.0102504,0.0176941,0.129178,0.0756949,0.02992,0.0513274,0.167115,0.329389,0.329389,0.0307214,0.0458836,0.143584,0.340861,0.0219057,0.176985,0.00561332,0.0103724,0.249232,0.0670528,0,0.0699919,0.00570304,0.237619,0.0536604,0.0357736,0.0476982,0.0609873,0.0303298,0.0215991,0.0571685,0.0232207,0.00751765,0.0496885,0.0965232,0.132059,0.00573534,0.0211043,0.0235515,0.0217109,0.0141343,0.0333928],[0.0150116,0.0128425,0.0107225,0.0117948,0.012577,0.0093789,0.0171561,0.00750414,0.0107225,0.00514682,0.00836359,0.0137249,0.0151172,0.0318918,0.00984593,0.0247049,0.0205161,0.0418364,0.0418364,0.0119864,0.0206623,0.0205161,0.0492193,0.0162672,0.0430669,0.00965029,0.00450347,0.0404924,0.0235446,0.00943278,0.044918,0.0130262,0.0479888,0.0274,0.0377311,0.0247049,0.0113379,0.0233573,0.0238041,0.0325344,0.00437381,0.0089836,0.00941785,0.0183565,0.0253752,0.00965029,0.0179795,0.0293798,0.00429866,0.00516905,0.0128425],["B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","IFNAR_change","B6_change","B6_change","IFNAR_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","IFNAR_change","IFNAR_change","IFNAR_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change"],[19.221668576301,17.2104341055091,15.3036138960131,13.5601281920846,11.0760117675121,10.332000554436,9.74632929395375,9.63097703401056,8.78184192119375,8.67687232116142,8.63030110275611,8.56334108080933,8.54510094461937,8.20800331119598,8.19721448354802,8.17922760262134,8.14555397955752,8.10366092684839,7.87326347391267,7.61367883601415,7.482806851125,6.99860109864935,6.92535245320433,6.90739647880397,6.84595362099431,6.54689133694428,6.26503562808235,6.1550315614782,5.75516254257877,5.6800116190561,5.65728661115811,5.52332990434662,5.44656253125729,5.4192700729927,5.39599958654797,5.39083339742318,5.37906490619956,5.37416567839605,5.3446255056902,5.34268343660864,5.30903262830347,5.28240349080547,5.27599186650881,5.25825729305696,5.20425454774741,5.20163642750632,5.19936594454796,5.10803341071076,5.05062042590016,5.04042328861203,5.01610278372591]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>virus<\/th>\n      <th>time<\/th>\n      <th>B6<\/th>\n      <th>IFNAR<\/th>\n      <th>RNaseL<\/th>\n      <th>cell_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,10]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>change_RNaseL_18S_graph &lt;- change_RNaseL_18S %&gt;% 
  ggplot(aes(x = start, y = fold_change, fill = cell_comp, width = 15)) +
  geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) + 
  scale_fill_brewer(palette=&quot;Set1&quot;) +
  facet_grid(virus ~ cell_comp) +
  theme_cowplot() +
  labs(x=&quot;Position&quot;, y=&quot;fold_change&quot;) +
  theme(axis.text.x = element_text(angle = 60, hjust = 1))
change_RNaseL_18S_graph</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-8-2.png" width="672" /></p>
</div>
<div id="s-1" class="section level2">
<h2>28s</h2>
<pre class="r"><code>change_RNaseL_28S &lt;- table_28S %&gt;% 
  filter(count &gt;= 20) %&gt;%
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count)
change_RNaseL_28S[is.na(change_RNaseL_28S)] &lt;- 0

change_RNaseL_28S &lt;- change_RNaseL_28S %&gt;% 
  mutate(B6_change = B6/RNaseL) %&gt;% 
  mutate(IFNAR_change = IFNAR/RNaseL) %&gt;% 
  gather(cell_comp, fold_change, B6_change:IFNAR_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_RNaseL_28S)</code></pre>
<div id="htmlwidget-4330c2b98d15ef8e110e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-4330c2b98d15ef8e110e">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133"],["r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|"],[2633,2832,1255,1255,2514,3418,3613,3454,203,1203,2240,206,1091,3418,2460,3680,2240,461,4122,3454,1566,390,1856,118,1341,2468,3615,2240,2241,1255,1867,3434,1812,353,2483,4136,2514,4123,4006,4113,3653,4172,67,1829,4155,2644,4121,4700,4705,2479,1404,394,2840,1868,1138,4138,4151,2516,4139,4114,4115,4154,169,2516,3418,1566,4137,4385,4125,391,4401,71,2633,1568,4142,2646,2657,4120,3521,1556,2485,4124,3679,4144,4132,4134,1846,1545,3768,4119,4006,426,203,4131,4700,206,4141,447,3774,82,2465,4133,1567,4118,4135,1558,3521,149,3869,3630,1145,4140,4157,406,1829,4158,2165,4156,4146,2461,4705,1767,430,2490,1355,4109,4398,1091,3767,2493,1395,1241,4160],[2634,2833,1256,1256,2515,3419,3614,3455,204,1204,2241,207,1092,3419,2461,3681,2241,462,4123,3455,1567,391,1857,119,1342,2469,3616,2241,2242,1256,1868,3435,1813,354,2484,4137,2515,4124,4007,4114,3654,4173,68,1830,4156,2645,4122,4701,4706,2480,1405,395,2841,1869,1139,4139,4152,2517,4140,4115,4116,4155,170,2517,3419,1567,4138,4386,4126,392,4402,72,2634,1569,4143,2647,2658,4121,3522,1557,2486,4125,3680,4145,4133,4135,1847,1546,3769,4120,4007,427,204,4132,4701,207,4142,448,3775,83,2466,4134,1568,4119,4136,1559,3522,150,3870,3631,1146,4141,4158,407,1830,4159,2166,4157,4147,2462,4706,1768,431,2491,1356,4110,4399,1092,3768,2494,1396,1242,4161],["nsp15","nsp15","nsp15","nsp15","nsp15","nsp15","nsp15","nsp15","nsp15","nsp15","nsp15","nsp15","mock","nsp15","nsp15","nsp15","nsp15","nsp15","ns2","nsp15","nsp15","mock","mock","mock","nsp15","mock","nsp15","ns2","nsp15","ns2","mock","nsp15","mock","nsp15","mock","ns2","nsp15","ns2","nsp15","ns2","nsp15","ns2","mock","MHVV","ns2","mock","ns2","mock","mock","mock","nsp15","mock","mock","mock","mock","ns2","ns2","nsp15","ns2","ns2","ns2","ns2","mock","nsp15","ns2","nsp15","ns2","nsp15","ns2","mock","mock","mock","ns2","nsp15","ns2","ns2","nsp15","ns2","mock","mock","mock","ns2","nsp15","ns2","ns2","ns2","nsp15","mock","mock","ns2","nsp15","mock","ns2","ns2","mock","ns2","ns2","mock","mock","mock","mock","ns2","nsp15","ns2","ns2","nsp15","MHVV","MHVV","ns2","mock","mock","ns2","ns2","mock","MHVS","ns2","MHVS","ns2","ns2","mock","mock","MHVS","mock","mock","nsp15","mock","mock","MHVS","nsp15","nsp15","nsp15","mock","mock"],["12","12","12","9","12","12","12","9","12","12","12","12","9","9","9","12","9","12","9","12","12","12","12","12","12","9","12","12","12","12","9","12","12","12","9","9","9","9","12","9","9","9","9","12","9","9","9","12","12","9","12","12","9","9","9","9","9","12","9","9","9","9","12","9","12","9","9","12","9","12","9","9","12","12","9","12","12","9","12","012","9","9","12","9","9","9","12","12","9","9","9","12","12","9","12","12","9","12","9","012","9","9","9","9","9","12","12","12","9","9","9","9","9","12","12","9","9","9","9","9","12","12","12","9","12","9","9","9","12","9","12","9","12"],[0.101087,0.117704,0.153535,0.138094,0.147996,0.138648,0.144707,0.132911,0.0818736,0.078931,0.185211,0.0879319,0.0999015,0.100887,0.133466,0.143841,0.111438,0.0602368,0.676617,0.172056,0.125493,0.0290415,0.104802,0.0220968,0.073392,0.055323,0.133975,0.0631143,0.0399848,0.0686926,0.0699539,0.0922592,0.119323,0.055044,0.0509795,0.315346,0.0858924,0.368925,0.0884512,0.101033,0.118842,0.258706,0.041378,0.168783,0.206659,0.0386347,0.353617,0.16541,0.270844,0.037263,0.0578135,0.0265162,0.0459501,0.0642387,0.12482,0.34137,0.134711,0.136744,0.231152,0.485266,0.254114,0.205128,0.270844,0.0879286,0.0293258,0.080339,1.01799,0.044139,0.358209,0.0694471,0.0983012,0.0665248,0.0309196,0.0659489,0.20819,0.0379323,0.0536592,0.649062,0.198871,0.129883,0.0404635,0.480674,0.0380807,0.468427,0.378109,0.840413,0.0427543,0.122479,0.0592093,0.66284,0.0610872,0,0.025182,0.307692,0.16541,0.0353823,0.722541,0.105433,0.0610382,0.152298,0.0381774,0.393418,0.0499805,0.544967,0.293915,0.0683722,0.0263039,0.0691694,0.0535783,0.029719,0.0848134,0.188289,0.881745,0.0593457,0.0900157,0.333716,0.0329639,0.275545,0.120934,0.138765,0.270844,0.0427155,0.203291,0.0653817,0.0221561,0.0418352,0.0518939,0.0222559,0.0432736,0.152718,0.111992,0.0393205,0.289784],[0.00500317,0.0025626,0.00964026,0.00694571,0.012691,0.008542,0.00549129,0.01683,0.00622346,0.00475912,0.0104945,0.00671157,0.00847503,0.00667856,0.01683,0.0025626,0.0122886,0.00561332,0.0852864,0.0156197,0.00927417,0.270394,0.281866,0.242535,0.00561332,0,0.00707766,0.00742743,0.00414897,0.0057134,0.0113,0.00890809,0.211399,0.00610143,0,0.0425136,0.00747999,0.0451059,0.0118368,0.0241083,0.00854856,0.0391436,0.00847503,0.0715521,0.0287744,0,0.0546974,0.206483,0.368719,0,0.0100063,0.217954,0.00918128,0.0137719,0.00741565,0.0489943,0.0171091,0.0187924,0.0360329,0.0987663,0.0458836,0.0313667,0.137655,0.0173643,0,0.00934999,0.15476,0.0059794,0.0487351,0.314641,0.0218938,0,0,0.008542,0.0290337,0,0.00561332,0.0930633,0,0.0695147,0,0.0645481,0,0.0935817,0,0.120023,0.00707766,0.545705,0.00776877,0.108617,0.0122886,0.14421,0,0.0388844,0.206483,0,0.139206,0.260562,0.0130657,0.050274,0,0,0.00961713,0.0767318,0.0311075,0.0165959,0.0763542,0.427872,0.00933225,0,0.0081219,0.0409582,0.136355,0.163875,0.0566816,0.0521051,0.0656341,0.0414767,0.0217753,0.0395501,0.368719,0.0183169,0.444102,0,0.00719969,0.0176563,0.0116532,0.010275,0.004271,0.04114,0.0274564,0,0.0688276],[0.00493237,0.00643353,0.00857804,0.00856168,0.00943584,0.00900694,0.00965029,0.00941785,0.00621908,0.00600463,0.0143682,0.00686243,0.00784308,0.00898976,0.0119864,0.0137249,0.0107021,0.00600463,0.0678261,0.017585,0.0130815,0.0283011,0.0295316,0.0258401,0.00793469,0.00601302,0.0147971,0.00703418,0.00450347,0.00781575,0.00810451,0.0107225,0.0246096,0.00643353,0.00601302,0.0381803,0.0107021,0.0471639,0.0113659,0.0130262,0.015411,0.0341377,0.00549015,0.0226757,0.0278491,0.00522872,0.0480622,0.0283011,0.0516802,0.00522872,0.00814914,0.030762,0.0065359,0.00915026,0.0180391,0.049859,0.0197639,0.0201584,0.0341377,0.0718688,0.0377311,0.0305442,0.0406059,0.0132706,0.00449406,0.0124144,0.158111,0.00686243,0.0556983,0.0492193,0.0154247,0.0104574,0.00488484,0.0105081,0.0332393,0.00605721,0.00857804,0.103761,0.0319925,0.0209211,0.0065359,0.0781573,0.00621908,0.0772589,0.062436,0.138797,0.00707688,0.0910556,0.00993456,0.111397,0.010274,0.0246096,0.00429866,0.052554,0.0283011,0.00605721,0.123974,0.0455278,0.0107189,0.0269852,0.00679733,0.070072,0.00898976,0.0988196,0.0534524,0.0124382,0.0140374,0.0788252,0.00988196,0.00549015,0.0156862,0.035036,0.165298,0.030762,0.0168988,0.0628852,0.0123726,0.0521049,0.0229082,0.026405,0.0516802,0.00815119,0.0849032,0.0125489,0.00428902,0.00810451,0.010196,0.00439026,0.00857804,0.030394,0.0223029,0.00784308,0.0578326],["B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","IFNAR_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","IFNAR_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change","B6_change"],[20.4946100961607,18.2953992598154,17.8986108714811,16.12931106979,15.6844541662428,15.3934632627729,14.9950934117006,14.112669027432,13.1649054200943,13.1450230905152,12.8903411700839,12.813522323725,12.7375342339999,11.2224353041683,11.134786090903,10.4802949383966,10.4127227366592,10.0317255184749,9.97576154312278,9.78424793858402,9.5931659213393,9.55418693973026,9.54455566240908,9.38599308826204,9.24951069291932,9.20053483939851,9.05413898669334,8.97251705244961,8.87866467412906,8.78899657742379,8.63147802889996,8.60426206574959,8.59010304921657,8.55580062578398,8.4781856704285,8.25939031385296,8.02575195522374,7.8221902768855,7.78215539464539,7.75613763031429,7.71150476932062,7.57830785319456,7.53677039789441,7.44334243264816,7.42067068594676,7.38894031426429,7.35748675674439,7.29593549367339,7.13462796196609,7.12660077418565,7.09442959625187,7.08517001495351,7.03041662204134,7.020423463377,6.91941393971983,6.84670771575844,6.81601303386477,6.78347487895865,6.77116501697537,6.75210939934993,6.7348685832112,6.71577582650716,6.67006518757126,6.62581948065649,6.52545804906921,6.47143639644284,6.43845146763982,6.43197817682658,6.43123757816666,6.39263459659117,6.37297321827977,6.36150477174059,6.32970578360806,6.27600612860555,6.26336896384701,6.26233860143531,6.25541498990445,6.25535605863475,6.21617566617176,6.20822996878749,6.19096069401307,6.15008450906057,6.12320471838278,6.06308140550797,6.05594528797489,6.05497957448648,6.04140525203197,5.99309652563928,5.9599317936577,5.95025000673268,5.94580494452015,5.85990832845719,5.8581046186486,5.85477794268752,5.84464914791298,5.84135270198656,5.82816558310614,5.72314058663059,5.69444625847802,5.64376028341461,5.61652884294274,5.61448224683183,5.55971460862136,5.51476630142198,5.49863055728087,5.49695293531218,5.43934061863308,5.42811182210765,5.42182927273537,5.41314900321485,5.40687993267968,5.37415800890513,5.33427506684896,5.32718938950653,5.32675101190617,5.30674944184005,5.30479446518921,5.28827423140626,5.27907037654639,5.25525468661238,5.24076919206969,5.2404004814021,5.23068624033016,5.21015387802915,5.16577213442698,5.16196537483451,5.08963318948607,5.06938085671464,5.04469552485183,5.0246101204185,5.02140977182339,5.01340034782254,5.01073788831904]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>virus<\/th>\n      <th>time<\/th>\n      <th>B6<\/th>\n      <th>IFNAR<\/th>\n      <th>RNaseL<\/th>\n      <th>cell_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,10]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>change_RNaseL_28S_graph &lt;- change_RNaseL_28S %&gt;% 
  ggplot(aes(x = start, y = fold_change, fill = cell_comp, width = 15)) +
  geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) + 
  scale_fill_brewer(palette=&quot;Set1&quot;) +
  facet_grid(virus ~ cell_comp) +
  theme_cowplot() +
  labs(x=&quot;Position&quot;, y=&quot;fold_change&quot;) +
  theme(axis.text.x = element_text(angle = 60, hjust = 1))
change_RNaseL_28S_graph</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-9-2.png" width="672" /></p>
</div>
<div id="s-2" class="section level2">
<h2>5s</h2>
<pre class="r"><code>change_RNaseL_5S &lt;- table_5S %&gt;% 
  filter(count &gt;= 20) %&gt;%
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count)
change_RNaseL_5S[is.na(change_RNaseL_5S)] &lt;- 0

change_RNaseL_5S &lt;- change_RNaseL_5S %&gt;% 
  mutate(B6_change = B6/RNaseL) %&gt;% 
  mutate(IFNAR_change = IFNAR/RNaseL) %&gt;% 
  gather(cell_comp, fold_change, B6_change:IFNAR_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_RNaseL_5S)</code></pre>
<div id="htmlwidget-2e4a86512140a68c1cf7" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2e4a86512140a68c1cf7">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8"],["r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240"],[53,113,119,53,119,52,90,53],[54,114,120,54,120,53,91,54],["MHVV","ns2","mock","nsp15","mock","ns2","mock","nsp15"],["12","9","012","12","012","9","9","9"],[0.0116906,0.087256,0.0469789,0.044139,0.0469789,0.356678,0.0528083,0.0568296],[0.262678,0.0225529,0.0546187,0.00317274,0.0546187,0.0585858,0,0],[0.0129576,0.00943278,0.00606408,0.00514682,0.00606408,0.0516557,0.00784308,0.010274],["IFNAR_change","B6_change","IFNAR_change","B6_change","B6_change","B6_change","B6_change","B6_change"],[20.2721182935111,9.25029524700035,9.00692273189008,8.5759750681003,7.74707787496207,6.90491078428905,6.73310740168403,5.53139964960093]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>virus<\/th>\n      <th>time<\/th>\n      <th>B6<\/th>\n      <th>IFNAR<\/th>\n      <th>RNaseL<\/th>\n      <th>cell_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,10]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="s-3" class="section level2">
<h2>5.8s</h2>
<pre class="r"><code>change_RNaseL_5.8S &lt;- table_5.8S %&gt;% 
  filter(count &gt;= 20) %&gt;%
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count)
change_RNaseL_5.8S[is.na(change_RNaseL_5.8S)] &lt;- 0

change_RNaseL_5.8S &lt;- change_RNaseL_5.8S %&gt;% 
  mutate(B6_change = B6/RNaseL) %&gt;% 
  mutate(IFNAR_change = IFNAR/RNaseL) %&gt;% 
  gather(cell_comp, fold_change, B6_change:IFNAR_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_RNaseL_5.8S)</code></pre>
<div id="htmlwidget-8fdf669a1f54f5366fd6" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-8fdf669a1f54f5366fd6">{"x":{"filter":"none","data":[[],[],[],[],[],[],[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>virus<\/th>\n      <th>time<\/th>\n      <th>B6<\/th>\n      <th>IFNAR<\/th>\n      <th>RNaseL<\/th>\n      <th>cell_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,10]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="fold-change-for-nsp15-activity-by-rrna" class="section level1">
<h1>Fold-change for Nsp15 activity by rRNA</h1>
<p>In this analysis, all positions in the MHV positive strand RNA with more than 20 reads were queried to identify sites with equal to or greater than a 5-fold decrease in signal comparing cells infected with WT/ns2 mutant virus to nsp15 mutant virus (MHV WT/nsp15 and ns2/nsp15).</p>
<p>This analysis is independent of cell type and just asks which sites change the most when nsp15 activity is lost. Thus, these sites are not independent of RNaseL actviity.</p>
<div id="s-4" class="section level2">
<h2>18s</h2>
<pre class="r"><code>change_nsp15_18S &lt;- table_18S %&gt;% 
  filter(count &gt;= 20) %&gt;%
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count)
change_nsp15_18S[is.na(change_nsp15_18S)] &lt;- 0

change_nsp15_18S &lt;- change_nsp15_18S %&gt;% 
  mutate(MHVS_change = MHVS/nsp15) %&gt;% 
  mutate(MHVV_change = MHVV/nsp15) %&gt;% 
  mutate(mock_change = mock/nsp15) %&gt;%
  mutate(ns2_change = ns2/nsp15) %&gt;% 
  gather(viral_comp, fold_change, MHVS_change:ns2_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_nsp15_18S)</code></pre>
<div id="htmlwidget-03badb59aca4dcc876b3" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-03badb59aca4dcc876b3">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40"],["r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|"],[770,71,72,771,1722,1725,1724,308,1723,75,1726,76,634,1291,1745,308,1769,77,1754,1768,877,631,1740,307,72,1808,78,1732,1864,1729,1739,1770,1805,1864,655,1282,74,1800,308,1096],[771,72,73,772,1723,1726,1725,309,1724,76,1727,77,635,1292,1746,309,1770,78,1755,1769,878,632,1741,308,73,1809,79,1733,1865,1730,1740,1771,1806,1865,656,1283,75,1801,309,1097],["IFNAR","B6","B6","IFNAR","B6","B6","B6","RNaseL","B6","B6","B6","B6","B6","RNaseL","B6","RNaseL","B6","B6","B6","B6","B6","B6","B6","RNaseL","IFNAR","B6","B6","B6","RNaseL","B6","B6","B6","B6","RNaseL","B6","RNaseL","B6","B6","RNaseL","B6"],["12","12","12","12","9","9","9","9","9","12","9","12","9","12","9","9","9","12","9","9","9","9","9","9","12","9","12","9","9","9","9","9","9","9","9","12","12","9","9","9"],[0.00302879,0.00492012,0.00447283,0,0.0230958,0.0174268,0.0348536,0.0818185,0.0178467,0.0468529,0.0132276,0.0175559,0,0.0194833,0.00503907,0.0818185,0,0.101645,0,0,0,0.00776857,0.0104981,0.0752331,0,0.0260352,0.0176677,0.0527003,0.0646566,0.117788,0.0065088,0,0.0249854,0.0646566,0.00587892,0.00675952,0.00626196,0.0153272,0.0818185,0.0138575],[0.062428,0.0141262,0.0126648,0.0504226,0.0235754,0.0200971,0.0394212,0.0281405,0.021643,0.0884102,0.0142998,0.0343412,0,0.016197,0,0.0281405,0,0.246721,0,0.00966206,0,0.011208,0.0146863,0.0218871,0.0211295,0.0390347,0.0333669,0.0529481,0.0166759,0.16696,0.00811613,0,0.0332375,0.0166759,0,0,0.0175359,0.0127539,0.0281405,0.0088891],[0,0.0947006,0.0782858,0,0.00983012,0.00960151,0.0203461,0.0478428,0.00777265,0.248115,0.00777265,0.0845992,0.00868708,0.130431,0.00480076,0.0478428,0,0.432466,0.00525797,0,0.0251468,0.012802,0.00960151,0.0460127,0,0.023318,0.0612397,0.0347483,0.0637903,0.130078,0.00731544,0,0.0185172,0.0637903,0,0.0369144,0.0366175,0.00708683,0.0478428,0],[0,0.0114753,0.00828774,0,0.202067,0.125526,0.261768,0.0619868,0.154612,0.0650269,0.134711,0.0221538,0.0535783,0.0252058,0.0382702,0.0619868,0.0382702,0.112522,0.0336778,0.071948,0,0.0581707,0.0750096,0.0390786,0,0.168389,0.0224725,0.254114,0.0197639,0.583238,0.0566399,0.0367394,0.157673,0.0197639,0.047455,0.00722957,0.0105191,0.0765404,0.0619868,0.0336778],[0.00414897,0.00657758,0.0058852,0.00451506,0.0205475,0.0135132,0.0299883,0.00941785,0.0181411,0.0309839,0.0177708,0.0115973,0.00758963,0.0193006,0.0057385,0.00941785,0.00592361,0.0675068,0.00536827,0.011477,0.0042576,0.00999609,0.013143,0.0132706,0.00378289,0.0303585,0.011078,0.0466484,0.0119864,0.108476,0.0105514,0.00684918,0.0294329,0.0119864,0.00907053,0.00707688,0.00709686,0.0149941,0.00941785,0.00666406],["MHVV_change","mock_change","mock_change","MHVV_change","ns2_change","ns2_change","ns2_change","MHVS_change","ns2_change","mock_change","ns2_change","mock_change","ns2_change","mock_change","ns2_change","ns2_change","ns2_change","mock_change","ns2_change","ns2_change","mock_change","ns2_change","ns2_change","MHVS_change","MHVV_change","ns2_change","mock_change","ns2_change","MHVS_change","ns2_change","ns2_change","ns2_change","ns2_change","mock_change","ns2_change","mock_change","mock_change","ns2_change","mock_change","ns2_change"],[15.0466260300749,14.3974835729858,13.3021477604839,11.1676478274929,9.83414040637547,9.28913950803659,8.72900431168156,8.6875985495628,8.52274669121498,8.00786860272593,7.58046908411552,7.29473239460909,7.0594086931774,6.75787281224418,6.66902500653481,6.58184192782854,6.46062114149986,6.40625833249391,6.27349220512381,6.26888559728152,5.90633220593762,5.81934536403734,5.70719013923762,5.66915587840791,5.58554438537732,5.54668379531268,5.52804657880484,5.44743228063556,5.39416338516986,5.37665474390649,5.36799855943287,5.36405817922729,5.35703243649114,5.32188980844958,5.23177807691502,5.21619696815546,5.15967625118714,5.1047011824651,5.08001295412435,5.05364597557645]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>cell<\/th>\n      <th>time<\/th>\n      <th>MHVS<\/th>\n      <th>MHVV<\/th>\n      <th>mock<\/th>\n      <th>ns2<\/th>\n      <th>nsp15<\/th>\n      <th>viral_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,9,10,12]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>change_nsp15_18S_graph &lt;- change_nsp15_18S %&gt;%
  ggplot(aes(x = start, y = fold_change, fill = viral_comp, width=5)) +
  geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) + 
  scale_fill_brewer(palette=&quot;Set1&quot;) +
  facet_grid(viral_comp ~ cell) +
  theme_cowplot() +
  labs(x=&quot;Position&quot;, y=&quot;fold_change&quot;) +
  theme(axis.text.x = element_text(angle = 60, hjust = 1))
change_nsp15_18S_graph</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-12-2.png" width="672" /></p>
<pre class="r"><code>interest_18S &lt;- ggplot(table_18S, aes(x = start, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts&quot;) +
    xlim(1720, 1726)
interest_18S</code></pre>
<pre><code>## Warning: Removed 58734 rows containing missing values (geom_bar).</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-12-3.png" width="672" /></p>
</div>
<div id="s-5" class="section level2">
<h2>28s</h2>
<pre class="r"><code>change_nsp15_28S &lt;- table_28S %&gt;% 
  filter(count &gt;= 20) %&gt;%
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count)
change_nsp15_28S[is.na(change_nsp15_28S)] &lt;- 0

change_nsp15_28S &lt;- change_nsp15_28S %&gt;% 
  mutate(MHVS_change = MHVS/nsp15) %&gt;% 
  mutate(MHVV_change = MHVV/nsp15) %&gt;% 
  mutate(mock_change = mock/nsp15) %&gt;%
  mutate(ns2_change = ns2/nsp15) %&gt;% 
  gather(viral_comp, fold_change, MHVS_change:ns2_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_nsp15_28S)</code></pre>
<div id="htmlwidget-98cf9d7128d9b7fdfa9a" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-98cf9d7128d9b7fdfa9a">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192"],["r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|"],[2832,1105,1119,1099,1079,71,4570,4121,1311,1317,73,1093,2489,4122,4123,71,1313,1545,1106,2840,4124,2379,1089,86,2480,1100,2485,1091,4119,71,1114,3822,4125,423,1107,4111,4120,4146,67,1938,75,4148,1111,2483,4145,4143,4132,4133,4137,2481,1203,4141,4173,4171,4160,4139,2479,4130,4140,1115,4131,2479,4134,1619,4138,4135,85,4114,1319,2490,4295,4172,2486,4154,381,4142,4127,1110,4128,2468,95,83,2441,382,71,2380,1933,4136,4158,3954,3634,1127,1199,4151,67,4144,2483,2378,1552,383,3305,1811,2377,4157,1131,1208,4109,1549,4129,1544,4118,1854,3306,1867,394,1293,1126,1085,4117,2100,4155,3300,1209,1807,4159,1841,1208,91,4126,4156,1197,2099,3768,1812,82,112,1829,4152,1138,2465,3765,91,407,1852,1130,3347,4112,902,901,2513,98,2485,2474,3952,2491,4168,1566,2472,3775,385,96,375,3953,4153,4012,4147,4167,917,1868,3766,1558,1116,4018,2456,1553,1809,4014,1554,1557,3767,2515,2465,4113,272,111,391,1203,1108,4169,2150,390,95],[2833,1106,1120,1100,1080,72,4571,4122,1312,1318,74,1094,2490,4123,4124,72,1314,1546,1107,2841,4125,2380,1090,87,2481,1101,2486,1092,4120,72,1115,3823,4126,424,1108,4112,4121,4147,68,1939,76,4149,1112,2484,4146,4144,4133,4134,4138,2482,1204,4142,4174,4172,4161,4140,2480,4131,4141,1116,4132,2480,4135,1620,4139,4136,86,4115,1320,2491,4296,4173,2487,4155,382,4143,4128,1111,4129,2469,96,84,2442,383,72,2381,1934,4137,4159,3955,3635,1128,1200,4152,68,4145,2484,2379,1553,384,3306,1812,2378,4158,1132,1209,4110,1550,4130,1545,4119,1855,3307,1868,395,1294,1127,1086,4118,2101,4156,3301,1210,1808,4160,1842,1209,92,4127,4157,1198,2100,3769,1813,83,113,1830,4153,1139,2466,3766,92,408,1853,1131,3348,4113,903,902,2514,99,2486,2475,3953,2492,4169,1567,2473,3776,386,97,376,3954,4154,4013,4148,4168,918,1869,3767,1559,1117,4019,2457,1554,1810,4015,1555,1558,3768,2516,2466,4114,273,112,392,1204,1109,4170,2151,391,96],["IFNAR","RNaseL","RNaseL","RNaseL","RNaseL","B6","RNaseL","B6","RNaseL","RNaseL","IFNAR","RNaseL","RNaseL","B6","B6","IFNAR","RNaseL","IFNAR","RNaseL","B6","B6","RNaseL","RNaseL","IFNAR","RNaseL","RNaseL","RNaseL","RNaseL","B6","IFNAR","RNaseL","RNaseL","B6","IFNAR","RNaseL","RNaseL","B6","B6","IFNAR","RNaseL","IFNAR","B6","RNaseL","RNaseL","B6","B6","B6","B6","B6","RNaseL","IFNAR","B6","B6","B6","B6","B6","B6","B6","B6","RNaseL","B6","RNaseL","B6","IFNAR","B6","B6","IFNAR","B6","RNaseL","RNaseL","B6","B6","RNaseL","B6","IFNAR","B6","B6","RNaseL","B6","RNaseL","IFNAR","IFNAR","RNaseL","IFNAR","B6","RNaseL","RNaseL","B6","B6","IFNAR","RNaseL","RNaseL","RNaseL","B6","B6","B6","B6","RNaseL","IFNAR","IFNAR","IFNAR","IFNAR","RNaseL","B6","RNaseL","RNaseL","RNaseL","IFNAR","B6","IFNAR","B6","IFNAR","IFNAR","B6","IFNAR","B6","IFNAR","RNaseL","B6","RNaseL","B6","IFNAR","RNaseL","IFNAR","B6","IFNAR","IFNAR","B6","B6","B6","RNaseL","RNaseL","RNaseL","IFNAR","IFNAR","IFNAR","IFNAR","B6","B6","RNaseL","RNaseL","IFNAR","IFNAR","IFNAR","RNaseL","IFNAR","RNaseL","B6","B6","RNaseL","IFNAR","B6","RNaseL","IFNAR","RNaseL","B6","IFNAR","B6","RNaseL","IFNAR","IFNAR","RNaseL","IFNAR","B6","RNaseL","B6","B6","B6","B6","RNaseL","IFNAR","RNaseL","RNaseL","RNaseL","IFNAR","IFNAR","RNaseL","IFNAR","IFNAR","RNaseL","RNaseL","B6","B6","B6","IFNAR","IFNAR","RNaseL","RNaseL","B6","IFNAR","IFNAR","B6"],["12","12","12","12","12","12","12","9","12","12","12","12","12","9","9","12","12","12","12","9","9","12","12","12","12","12","12","12","9","12","12","12","9","12","12","12","9","9","12","12","12","9","12","12","9","9","9","9","9","12","12","9","9","9","9","9","9","9","9","12","9","12","9","12","9","9","12","9","12","12","9","9","12","9","12","9","9","12","9","12","12","12","12","12","12","12","12","9","9","12","9","12","12","9","12","9","9","12","12","12","12","12","12","9","12","12","12","12","9","12","9","12","12","9","12","12","12","12","9","12","9","12","12","12","9","12","12","12","9","9","12","12","12","12","12","12","12","9","12","12","12","12","12","12","12","12","12","12","12","12","12","9","12","12","12","9","12","9","12","12","12","12","12","9","12","9","9","9","9","12","12","12","12","12","12","12","12","12","12","12","12","9","9","9","12","12","12","12","9","12","12","12"],[0.00288456,0.00596429,0,0.004175,0.00397619,0.0390255,0.00516905,0.0237256,0.0244536,0,0,0,0.00477143,0.0482911,0.027295,0.0103844,0.00497024,0.0555278,0.0127238,0.00860842,0.0377931,0.0063619,0.00536786,0.00317301,0,0.0061631,0.00536786,0.0119286,0.0787355,0.0103844,0.00835,0.00457262,0.0356934,0.108459,0.0286286,0.0151095,0.0602589,0.00986819,0.00735562,0,0.00389415,0.0256153,0.00795238,0,0,0.0390528,0.0428321,0.0403126,0.155161,0.00397619,0.00389415,0.0900734,0.00566896,0.0130176,0.223399,0.0289747,0,0.043672,0.0291846,0.019881,0.0312842,0,0.116319,0,0.0432521,0.0304444,0.0203361,0.059839,0.0129226,0.0127238,0,0.0180567,0,0.0268751,0.00346147,0.0191065,0.263921,0.00715714,0.070967,0.00735595,0.00302879,0.00649026,0.00516905,0.0098075,0.0390255,0.00735595,0.00516905,0.0413624,0.0449317,0.110046,0.0181597,0.00497024,0.0186881,0.0146973,0.0080511,0.0522804,0.00419923,0.00974167,0.0121151,0.0194708,0.0312975,0.00649026,0.0186881,0.16209,0.012525,0.0204774,0.0196821,0.068364,0.05417,0.00519221,0.081675,0.00533643,0.00605757,0.0140674,0.0333167,0.0026837,0,0,0.285757,0.0270381,0.027295,0.0317301,0.00695833,0.00850945,0.0802053,0.0216342,0.0106729,0.00883384,0.0459815,0.0512306,0,0.00497024,0.0153083,0.0444222,0.0245187,0.0673544,0.0566816,0.0302344,0.0252715,0.00815119,0.00675952,0.00576912,0.241149,0.00692294,0.00795238,0.00605757,0.0294238,0.0109584,0.0107348,0.0117298,0.00519221,0.00713869,0.0061631,0.0258168,0.0528833,0.0100781,0.010096,0.00566896,0.0359845,0.0062018,0.00475952,0.0139167,0.0317301,0.127237,0.0823071,0.0361134,0.00839846,0.005459,0.0151172,0.00397619,0.0124036,0.0103381,0.0123262,0,0.00677871,0.0398069,0.0184893,0.00374993,0.019615,0.00556667,0.0572571,0.0102881,0.0172168,0,0.0738447,0.0458645,0.00536786,0.0276345,0.0184766,0.155622,0.0428357,0.00760381],[0.0979639,0.010798,0,0,0,0.112522,0,0.0293727,0.026455,0,0.0096043,0,0,0.0479238,0.0285997,0.121014,0,0.114771,0,0.0200971,0.0463779,0,0,0,0,0,0,0.0134975,0,0.121014,0,0,0.0363293,0.238667,0.042652,0.0151172,0.0722722,0.00927558,0.0739531,0,0.0120054,0.0371023,0,0,0.00850261,0.0394212,0.0506292,0.0409671,0.139907,0,0.0374568,0.11208,0.00772965,0.0088891,0.325418,0.0200971,0,0.0475373,0.0285997,0.0172768,0.0363293,0,0.13179,0.0220899,0.032078,0.0251214,0.0782751,0.120969,0.0129576,0,0.00850261,0.0228025,0,0.0255078,0,0.0231889,0.284065,0,0.0637696,0,0.017768,0.0129658,0,0.0110449,0.112522,0.0134975,0,0.0262808,0.0479238,0.288129,0.0156336,0,0.0507505,0.0127539,0.0172924,0.0846396,0,0.0156571,0,0.0211295,0.0657895,0,0.0361732,0.180874,0.010798,0.069107,0.0259151,0.143104,0.0564264,0.0120054,0,0.0110449,0,0.0142998,0.0297733,0.00560175,0.0168075,0,0.37141,0.0496707,0.0340105,0.0672301,0.0156571,0,0.0850261,0.0187284,0.0172877,0.0250861,0.0479238,0.0479238,0,0,0.0118778,0.0494622,0.0970035,0.103246,0.0715521,0.0371023,0.0143697,0.010798,0,0.0192086,0.4466,0,0,0,0.0458914,0.0112035,0.015831,0,0.0105647,0.00927558,0,0.051383,0.037253,0.0204836,0.0499424,0.00850261,0.0383328,0,0.0115252,0.0431919,0.0547445,0.13295,0.167369,0.0382618,0.011208,0.010435,0.0127539,0,0.0196888,0.0118778,0.0145773,0,0,0.04466,0.0275348,0,0.013446,0,0.0518303,0.0100485,0.0270538,0.00811613,0.240108,0.0739531,0.0118778,0.0340136,0.0266673,0.425951,0.058106,0.0219199],[0,0.136583,0.142736,0.100899,0.097208,0.247484,0.120587,0.0137164,0.355609,0.0504497,0.0344138,0.0713679,0.099669,0.0290332,0.0132592,0.106519,0.0590631,0.545705,0.107052,0.0459501,0.0182886,0.0627546,0.130431,0.0475238,0.0590631,0.0430669,0.0935166,0.263323,0.0315478,0.106519,0.0590631,0.0492193,0.0276615,0.971781,0.171037,0.130431,0.0326909,0.00480076,0,0.0627546,0.0475238,0.0171456,0.066446,0.0590631,0,0.0173742,0.0226321,0.03612,0.137393,0.0676765,0,0.0756691,0,0.00845848,0.203689,0,0.037263,0.0230894,0,0.135353,0.016917,0.0652155,0.126649,0,0.0148595,0.024461,0.257284,0.107446,0.097208,0.120587,0,0.00983012,0.0541412,0.0251468,0.0426076,0.0118876,0.238666,0.0541412,0.0416066,0.0799813,0.0589951,0.0622726,0.0590631,0.109796,0.247484,0.0762899,0.0406059,0.0240038,0.030862,0.68336,0.0598688,0.0455278,0.242405,0.0153167,0.0347235,0.0416066,0.0509795,0.10213,0.0852152,0.145849,0.211399,0.0327751,0.139044,0.166426,0.217795,0.233791,0.10213,0.467045,0.0171456,0.0508014,0.080927,0.0344138,0.0344138,0.0699539,0.217954,0.0290415,0,0.033223,0.355942,0.318695,0.0265185,0.167153,0.0442973,0.0458851,0.0667534,0.10488,0.0770214,0,0.0333767,0.0457215,0.0270706,0.0258401,0.110743,0.211399,0.262201,0.334306,0.226148,0.0324623,0.0555577,0.0578326,0.0418364,0.0557176,1.21595,0.0393301,0.0713679,0.0344138,0.248557,0.063765,0.0808111,0.0762899,0.0540789,0.0404635,0.0246096,0.124545,0.461431,0.00640101,0,0.0278901,0.262093,0.0376913,0.0344138,0.0319925,0.162237,0.166426,0.548795,0.0144023,0.00822987,0.0262899,0.0642387,0.0258401,0.0868539,0.0381449,0.0492193,0.0615241,0.0540789,0.178624,0.0836727,0.0360526,0.126184,0.0442973,0.497115,0.0381774,0.037263,0.0352056,0.373636,0.314641,0.030762,0.125509,0.0105159,0.665334,0.270394,0],[0,0.0095743,0.00918351,0.00468945,0.00586181,0.0404824,0.00547103,0.353617,0.0414235,0.00508024,0,0.00527563,0.0109421,0.676617,0.368925,0.0142835,0.00390788,0.0694179,0.00859733,0,0.480674,0.00703418,0.00996508,0,0,0,0.00762036,0.0273551,0.66284,0.0142835,0.00410327,0.00664339,0.358209,0.177116,0.0212979,0.019344,0.649062,0.120934,0.00657041,0,0,0.232683,0.00605721,0.00996508,0.0413318,0.289323,0.378109,0.393418,1.01799,0.00840193,0,0.722541,0.0551091,0.0811328,1.40528,0.231152,0,0.352086,0.188289,0.0125052,0.307692,0.00586181,0.840413,0,0.34137,0.293915,0.0414222,0.485266,0.0123098,0.0195394,0.0336778,0.258706,0.00508024,0.205128,0,0.20819,1.39303,0.00488484,0.451588,0.0144591,0.00599907,0,0.00820654,0.00742743,0.0404824,0.00644799,0.00801115,0.315346,0.333716,0.190256,0.0152721,0.00508024,0.0336077,0.134711,0.0103597,0.468427,0,0.00879272,0.00685609,0.00999846,0.0448502,0,0.0148499,0.881745,0.0195394,0.023838,0.0220795,0.10427,0.310754,0,0.544967,0,0,0,0,0,0,0.00429866,1.45733,0.0543195,0.206659,0.0477069,0.00449406,0.00599907,0.440873,0.0105698,0.0131408,0.00765022,0.293915,0.275545,0,0,0.0252058,0.0628474,0.0505636,0.0525633,0.0791306,0.16992,0.012591,0.0134822,0.00488484,0.0105698,0.291098,0.00599907,0.00918351,0,0.0572504,0.0113159,0.0149817,0.0093789,0.00599907,0,0,0.0282814,0.0912489,0.0658247,0.00799877,0,0.0636984,0,0,0.0113328,0.0425649,0.676617,0.0945706,0.20819,0.0535783,0,0,0,0.0108555,0.00801115,0.00820654,0.0117236,0.00828444,0.0422792,0.019344,0,0.0117125,0.00781575,0.080893,0,0.101033,0,0.13198,0.0439932,0.00664339,0.0205163,0.102564,0.246533,0.0254246,0.00908464],[0.0025626,0.00729133,0.00836359,0.00600463,0.00643353,0.016444,0.00836359,0.0272116,0.0298087,0.00428902,0.00292869,0.00643353,0.00900694,0.0644193,0.0351714,0.0118368,0.00579018,0.0545468,0.0107225,0.00481293,0.0505358,0.00664798,0.0139393,0.0051252,0.00643353,0.00471792,0.0102936,0.0291653,0.0734898,0.0118368,0.00664798,0.00557573,0.0416504,0.113243,0.0199439,0.015226,0.0764516,0.0142537,0.00878606,0.00750578,0.00573534,0.0283223,0.00814914,0.00729133,0.00518316,0.0364672,0.0477591,0.0497954,0.129024,0.00857804,0.00475912,0.091816,0.00703429,0.0103663,0.18067,0.0298032,0.00481293,0.0455378,0.0244349,0.017585,0.0399844,0.00857804,0.111253,0.00292869,0.0453526,0.039429,0.0346561,0.0662704,0.013296,0.0165127,0.00462782,0.0357268,0.00750578,0.0285074,0.0059794,0.0294329,0.197145,0.00772024,0.0651597,0.0115804,0.008542,0.00903012,0.00857804,0.0159857,0.016444,0.0111515,0.00600463,0.0468336,0.0496102,0.102382,0.00898976,0.00686243,0.0368856,0.0205475,0.00536592,0.0729345,0.00795985,0.0160838,0.0135452,0.0233075,0.0338019,0.00524723,0.0223029,0.142352,0.03517,0.0379578,0.0167272,0.077,0.0512763,0.00841997,0.0907053,0.00573534,0.00573534,0.0116621,0.0363645,0.00484664,0.00280666,0.00557573,0.245645,0.0538272,0.0349863,0.0283106,0.00750578,0.00780983,0.0753409,0.0179382,0.0131791,0.00432736,0.0507209,0.0479442,0.00471792,0.00450347,0.0193006,0.0368526,0.0461268,0.0589398,0.0399034,0.0299883,0.00986637,0.0102936,0.00750578,0.0100063,0.218431,0.00707766,0.0128671,0.00622346,0.0450347,0.0115973,0.014713,0.0139393,0.00988432,0.00740451,0.00450347,0.0228193,0.0849226,0.0122174,0.00927417,0.00518316,0.0488948,0.00707766,0.00646752,0.00814914,0.0306292,0.127913,0.103794,0.039429,0.0101812,0.00499805,0.0122174,0.00493237,0.0165959,0.00729133,0.00943584,0.0117948,0.0103724,0.03429,0.0160838,0.00695563,0.0244057,0.00857804,0.0962885,0.00740451,0.019622,0.00684918,0.0728511,0.0613804,0.00600463,0.0246619,0.0201773,0.130937,0.0533265,0.00432736],["MHVV_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","ns2_change","mock_change","mock_change","mock_change","mock_change","mock_change","ns2_change","ns2_change","MHVV_change","mock_change","mock_change","mock_change","mock_change","ns2_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","ns2_change","mock_change","mock_change","mock_change","ns2_change","mock_change","mock_change","mock_change","ns2_change","ns2_change","MHVV_change","mock_change","mock_change","ns2_change","mock_change","mock_change","ns2_change","ns2_change","ns2_change","ns2_change","ns2_change","mock_change","MHVV_change","ns2_change","ns2_change","ns2_change","ns2_change","ns2_change","mock_change","ns2_change","ns2_change","mock_change","ns2_change","mock_change","ns2_change","MHVV_change","ns2_change","ns2_change","mock_change","ns2_change","mock_change","mock_change","ns2_change","ns2_change","mock_change","ns2_change","mock_change","ns2_change","ns2_change","mock_change","ns2_change","mock_change","mock_change","mock_change","mock_change","mock_change","MHVV_change","mock_change","mock_change","ns2_change","ns2_change","mock_change","mock_change","mock_change","mock_change","ns2_change","mock_change","ns2_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","ns2_change","mock_change","mock_change","mock_change","mock_change","ns2_change","mock_change","ns2_change","mock_change","mock_change","mock_change","mock_change","mock_change","MHVV_change","mock_change","ns2_change","mock_change","ns2_change","mock_change","mock_change","mock_change","ns2_change","mock_change","mock_change","MHVV_change","ns2_change","ns2_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","ns2_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","ns2_change","MHVV_change","mock_change","mock_change","mock_change","mock_change","MHVV_change","mock_change","ns2_change","mock_change","ns2_change","ns2_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","ns2_change","mock_change","mock_change","mock_change","mock_change","mock_change","ns2_change","mock_change","mock_change","MHVV_change"],[38.2283227971591,18.7322477517819,17.0663554765358,16.8035332734906,15.1095899140907,15.0501094624179,14.4180907959381,12.9950829793176,11.929705086099,11.7625238399448,11.7505779034312,11.0931168425421,11.0658003717134,10.5033274189567,10.4893464576332,10.2235401459854,10.2005637130452,10.004344892826,9.98386570296106,9.54721967699509,9.51155418534979,9.43964933709187,9.35706958025152,9.27257472879107,9.18051209833482,9.12836589005324,9.08492655630683,9.02864019914076,9.01948297586876,8.99896931603136,8.88436788317654,8.82741811386132,8.60037358584792,8.58137809842551,8.5759054146882,8.56633390253514,8.48984193921383,8.48439352589152,8.41709480700109,8.36083658194085,8.28613473656313,8.21554040455754,8.15374383063734,8.10045629535352,7.97424737033007,7.93378707441207,7.91700429865722,7.90068962193295,7.88992745535714,7.88950622752983,7.87053068634537,7.86944541256426,7.83435144129685,7.8265919373354,7.77815907455582,7.7559456702636,7.74226926217502,7.73173056230209,7.70574055960941,7.69707136764288,7.69530116745531,7.60261085282885,7.55407045203275,7.54258730012395,7.52702160405357,7.45428491719293,7.42391671307504,7.32251502933436,7.31107099879663,7.30268217795999,7.27724933121859,7.24123067277226,7.21326764173743,7.19560535159292,7.12573167876376,7.07337707123661,7.06601739836161,7.01289079095987,6.93048003597316,6.90660944354254,6.90647389370171,6.89609883368106,6.88538407375111,6.86838862233121,6.84273899294575,6.84122315383581,6.76243165690475,6.73332820880735,6.72676183526775,6.67461077142466,6.65966610899512,6.63435546883538,6.57180579955321,6.5560773816766,6.47111772072636,6.42257093693657,6.40458048832579,6.34986756860941,6.29117325694711,6.25759948514427,6.25405672462199,6.24617178968713,6.23434620609876,6.1941173991233,6.19263576912141,6.15923472909389,6.10562437227988,6.06551948051948,6.06038267191666,6.0334419243774,6.0081053698075,6.00030686933992,6.00030686933992,5.99839651520738,5.99359265217451,5.99208936500338,5.98843465186378,5.95850229476678,5.93266706018848,5.92070551691338,5.90685496894499,5.9042549433781,5.90175837820986,5.87530074278185,5.85170870005535,5.84674047563301,5.84420787458931,5.79709106707092,5.79475127610117,5.74720195560673,5.73782514328348,5.7378199477292,5.73780089738143,5.73633882005612,5.68435269734731,5.67199074309719,5.66738673897462,5.66620982183051,5.63101728396563,5.61830652055646,5.57389105462723,5.56825200123922,5.56674647829292,5.55693548432674,5.54654117866497,5.52968927252686,5.51923294703862,5.49826252662258,5.49249643172704,5.4730079702711,5.47118061738187,5.46471002132484,5.46458619686597,5.45787995249635,5.43354772463396,5.38778299801922,5.38510723870708,5.38090662838886,5.36034506736913,5.32539003003818,5.32101949433477,5.30017891458485,5.29680827445705,5.28966563210932,5.28734801626298,5.28012376677065,5.26247397163399,5.26003141225078,5.25796814379492,5.23888110583756,5.23345525099573,5.23154211920185,5.21620756604605,5.21620544646794,5.21373066985461,5.20921551472733,5.20229672092416,5.18322567474118,5.17026760142098,5.16403514089466,5.16276606240621,5.15596575600546,5.1489654469473,5.14011896314595,5.1287626405092,5.12608259314048,5.12304671561778,5.08918615354048,5.08313798179142,5.08132918884654,5.07053716257396,5.06542094949346]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>cell<\/th>\n      <th>time<\/th>\n      <th>MHVS<\/th>\n      <th>MHVV<\/th>\n      <th>mock<\/th>\n      <th>ns2<\/th>\n      <th>nsp15<\/th>\n      <th>viral_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,9,10,12]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>change_nsp15_28S_graph &lt;- change_nsp15_28S %&gt;%
  ggplot(aes(x = start, y = fold_change, fill = viral_comp, width=10)) +
  geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) + 
  scale_fill_brewer(palette=&quot;Set1&quot;) +
  facet_grid(viral_comp ~ cell) +
  theme_cowplot() +
  labs(x=&quot;Position&quot;, y=&quot;fold_change&quot;) +
  theme(axis.text.x = element_text(angle = 60, hjust = 1))
change_nsp15_28S_graph</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-13-2.png" width="672" /></p>
<pre class="r"><code>interest_28S &lt;- ggplot(table_28S, aes(x = start, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) +
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts&quot;) +
    xlim(4120, 4126)
interest_28S</code></pre>
<pre><code>## Warning: Removed 114669 rows containing missing values (geom_bar).</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-13-3.png" width="672" /></p>
</div>
</div>
<div id="s-6" class="section level1">
<h1>5s</h1>
<pre class="r"><code>change_nsp15_5S &lt;- table_5S %&gt;% 
  filter(count &gt;= 20) %&gt;%
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count)
change_nsp15_5S[is.na(change_nsp15_5S)] &lt;- 0

change_nsp15_5S &lt;- change_nsp15_5S %&gt;% 
  mutate(MHVS_change = MHVS/nsp15) %&gt;% 
  mutate(MHVV_change = MHVV/nsp15) %&gt;% 
  mutate(mock_change = mock/nsp15) %&gt;%
  #mutate(ns2_change = ns2/nsp15) %&gt;% 
  gather(viral_comp, fold_change, MHVS_change:mock_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))


datatable(change_nsp15_5S)</code></pre>
<div id="htmlwidget-f6d623932aa9979b5db9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-f6d623932aa9979b5db9">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19"],["r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240"],[53,54,51,52,55,72,74,83,56,113,75,73,71,76,51,75,52,77,72],[54,55,52,53,56,73,75,84,57,114,76,74,72,77,52,76,53,78,73],["IFNAR","IFNAR","IFNAR","IFNAR","IFNAR","RNaseL","RNaseL","RNaseL","IFNAR","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","B6","RNaseL","RNaseL","B6"],["12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12"],[0.00317301,0,0,0.00663448,0,0.0210738,0.0153083,0,0.00389415,0.0101393,0.0288274,0.00695833,0.00477143,0.0121274,0,0.0185623,0.02505,0.0337976,0.00838656],[0.262678,0.0638686,0.0451402,0.132539,0.0480215,0.0442717,0.0350934,0,0.0734729,0.0237555,0.0718065,0.0183565,0,0.0188964,0.0194363,0.0182666,0.0939423,0.0361732,0.00657597],[0,0,0,0,0,0.308851,0.233791,0.0430669,0,0.0824423,0.394985,0.104591,0.0602936,0.0898251,0.0381449,0.056189,0.155041,0.195647,0.0372489],[0.00657041,0,0.00742743,0.030281,0.00628474,0.0265736,0.0201256,0,0.0159975,0.00703418,0.0398603,0.0148499,0.00586181,0.0168039,0.00683878,0.0153004,0.03224,0.0369294,0.00796898],[0.00317274,0.00280666,0.00292869,0.0117147,0.004271,0.0330255,0.0255197,0.00493237,0.00878606,0.0100792,0.0484659,0.0128671,0.00814914,0.0145827,0.00643353,0.00969328,0.030452,0.0390301,0.00744305],["MHVV_change","MHVV_change","MHVV_change","MHVV_change","MHVV_change","mock_change","mock_change","mock_change","MHVV_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change"],[82.7921607191261,22.7560873066207,15.4131027865701,11.313904752149,11.2436197611801,9.35189474799776,9.16119703601531,8.73148202588208,8.36244004707457,8.17944876577506,8.14975064942568,8.12856043708373,7.39876845900304,6.15970293567035,5.92907781575589,5.79669626792995,5.09132405096545,5.01272095126582,5.00452099609703]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>cell<\/th>\n      <th>time<\/th>\n      <th>MHVS<\/th>\n      <th>MHVV<\/th>\n      <th>mock<\/th>\n      <th>ns2<\/th>\n      <th>nsp15<\/th>\n      <th>viral_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,9,10,12]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="s-7" class="section level1">
<h1>5.8S</h1>
<pre class="r"><code>change_nsp15_5.8S &lt;- table_5.8S %&gt;% 
  filter(count &gt;= 20) %&gt;%
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count)
change_nsp15_5.8S[is.na(change_nsp15_5.8S)] &lt;- 0

change_nsp15_5.8S &lt;- change_nsp15_5.8S %&gt;% 
  mutate(MHVS_change = MHVS/nsp15) %&gt;% 
  mutate(MHVV_change = MHVV/nsp15) %&gt;% 
  mutate(mock_change = mock/nsp15) %&gt;%
  #mutate(ns2_change = ns2/nsp15) %&gt;% 
  gather(viral_comp, fold_change, MHVS_change:mock_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_nsp15_5.8S)</code></pre>
<div id="htmlwidget-73f2b41ab3d7d6e041cb" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-73f2b41ab3d7d6e041cb">{"x":{"filter":"none","data":[["1"],["r5.8s|NR_003280.2"],[72],[73],["IFNAR"],["12"],[0.00302879],[0.0350557],[0],[0],[0.004271],["MHVV_change"],[8.20784359634746]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>cell<\/th>\n      <th>time<\/th>\n      <th>MHVS<\/th>\n      <th>MHVV<\/th>\n      <th>mock<\/th>\n      <th>ns2<\/th>\n      <th>nsp15<\/th>\n      <th>viral_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,9,10,12]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<div id="fold-change-for-nsp15-muant-and-rnasel-ko-reads" class="section level2">
<h2>Fold-change for Nsp15 muant and RNaseL KO reads</h2>
<p>Same analysis as above focused in RNaseL KO cells.</p>
<pre class="r"><code>#18s 

change_nsp15RNaseL_18S &lt;- change_nsp15_18S %&gt;% 
  mutate(MHVS_change = MHVS/nsp15) %&gt;% 
  mutate(MHVV_change = MHVV/nsp15) %&gt;% 
  mutate(mock_change = mock/nsp15) %&gt;%
  mutate(ns2_change = ns2/nsp15) %&gt;% 
  gather(viral_comp, fold_change, MHVS_change:ns2_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  filter(cell == &quot;RNaseL&quot;) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_nsp15RNaseL_18S)</code></pre>
<div id="htmlwidget-b21a1f9021c365440e2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-b21a1f9021c365440e2e">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8"],["r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|","r18s|gi|374088232|ref|NR_003278.3|"],[308,1291,308,307,1864,1864,1282,308],[309,1292,309,308,1865,1865,1283,309],["RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL"],["9","12","9","9","9","9","12","9"],[0.0818185,0.0194833,0.0818185,0.0752331,0.0646566,0.0646566,0.00675952,0.0818185],[0.0281405,0.016197,0.0281405,0.0218871,0.0166759,0.0166759,0,0.0281405],[0.0478428,0.130431,0.0478428,0.0460127,0.0637903,0.0637903,0.0369144,0.0478428],[0.0619868,0.0252058,0.0619868,0.0390786,0.0197639,0.0197639,0.00722957,0.0619868],[0.00941785,0.0193006,0.00941785,0.0132706,0.0119864,0.0119864,0.00707688,0.00941785],["MHVS_change","mock_change","ns2_change","MHVS_change","MHVS_change","mock_change","mock_change","mock_change"],[8.6875985495628,6.75787281224418,6.58184192782854,5.66915587840791,5.39416338516986,5.32188980844958,5.21619696815546,5.08001295412435]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>cell<\/th>\n      <th>time<\/th>\n      <th>MHVS<\/th>\n      <th>MHVV<\/th>\n      <th>mock<\/th>\n      <th>ns2<\/th>\n      <th>nsp15<\/th>\n      <th>viral_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,9,10,12]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>change_nsp15RNaseL_18S_graph &lt;- change_nsp15RNaseL_18S %&gt;%
  ggplot(aes(x = start, y = fold_change, fill = viral_comp, width=10)) +
  geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) + 
  scale_fill_brewer(palette=&quot;Set1&quot;) +
  facet_grid(viral_comp ~ cell) +
  theme_cowplot() +
  labs(x=&quot;Position&quot;, y=&quot;fold_change&quot;) +
  theme(axis.text.x = element_text(angle = 60, hjust = 1))
change_nsp15RNaseL_18S_graph</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-16-2.png" width="672" /></p>
<pre class="r"><code>#28s

change_nsp15RNaseL_28S &lt;- change_nsp15_28S %&gt;% 
  mutate(MHVS_change = MHVS/nsp15) %&gt;% 
  mutate(MHVV_change = MHVV/nsp15) %&gt;% 
  mutate(mock_change = mock/nsp15) %&gt;%
  mutate(ns2_change = ns2/nsp15) %&gt;% 
  gather(viral_comp, fold_change, MHVS_change:ns2_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  filter(cell == &quot;RNaseL&quot;) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_nsp15RNaseL_28S)</code></pre>
<div id="htmlwidget-ee95c57288d1143e0610" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-ee95c57288d1143e0610">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68"],["r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|","r28s|gi|120444900|ref|NR_003279.1|"],[1105,1119,1099,1079,4570,1311,1317,1093,2489,1313,1106,2379,1089,2480,1100,2485,1091,1114,3822,1107,4111,1938,1111,2483,2481,1115,2479,1319,2490,2486,1110,2468,2441,2380,1933,3634,1127,1199,2378,2377,1131,1208,4109,1085,2100,1209,1197,2099,3768,2465,3765,1130,4112,2513,2474,2491,3775,375,4012,3766,1116,4018,2456,4014,3767,2515,1203,1108],[1106,1120,1100,1080,4571,1312,1318,1094,2490,1314,1107,2380,1090,2481,1101,2486,1092,1115,3823,1108,4112,1939,1112,2484,2482,1116,2480,1320,2491,2487,1111,2469,2442,2381,1934,3635,1128,1200,2379,2378,1132,1209,4110,1086,2101,1210,1198,2100,3769,2466,3766,1131,4113,2514,2475,2492,3776,376,4013,3767,1117,4019,2457,4015,3768,2516,1204,1109],["RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL"],["12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","9","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","12"],[0.00596429,0,0.004175,0.00397619,0.00516905,0.0244536,0,0,0.00477143,0.00497024,0.0127238,0.0063619,0.00536786,0,0.0061631,0.00536786,0.0119286,0.00835,0.00457262,0.0286286,0.0151095,0,0.00795238,0,0.00397619,0.019881,0,0.0129226,0.0127238,0,0.00715714,0.00735595,0.00516905,0.00735595,0.00516905,0.0181597,0.00497024,0.0186881,0.00974167,0.0186881,0.012525,0.0204774,0.0196821,0,0.0270381,0.00695833,0,0.00497024,0.0153083,0.00815119,0.00675952,0.00795238,0.0294238,0.0117298,0.0061631,0.0528833,0.0359845,0.0139167,0.0823071,0.00397619,0.0103381,0.0123262,0,0.0184893,0.00556667,0.0572571,0.00536786,0.0276345],[0.010798,0,0,0,0,0.026455,0,0,0,0,0,0,0,0,0,0,0.0134975,0,0,0.042652,0.0151172,0,0,0,0,0.0172768,0,0.0129576,0,0,0,0,0,0.0134975,0,0.0156336,0,0.0507505,0.0156571,0.0361732,0.010798,0.069107,0.0259151,0,0.0496707,0.0156571,0,0,0.0118778,0.010798,0,0,0.0458914,0,0,0.037253,0.0383328,0.0431919,0.167369,0,0.0118778,0.0145773,0,0.0275348,0,0.0518303,0.0118778,0.0340136],[0.136583,0.142736,0.100899,0.097208,0.120587,0.355609,0.0504497,0.0713679,0.099669,0.0590631,0.107052,0.0627546,0.130431,0.0590631,0.0430669,0.0935166,0.263323,0.0590631,0.0492193,0.171037,0.130431,0.0627546,0.066446,0.0590631,0.0676765,0.135353,0.0652155,0.097208,0.120587,0.0541412,0.0541412,0.0799813,0.0590631,0.0762899,0.0406059,0.0598688,0.0455278,0.242405,0.10213,0.139044,0.217795,0.233791,0.10213,0.033223,0.318695,0.0442973,0.0270706,0.0258401,0.110743,0.0578326,0.0418364,0.0713679,0.248557,0.0762899,0.0246096,0.461431,0.262093,0.0319925,0.548795,0.0258401,0.0381449,0.0492193,0.0615241,0.0836727,0.0442973,0.497115,0.030762,0.125509],[0.0095743,0.00918351,0.00468945,0.00586181,0.00547103,0.0414235,0.00508024,0.00527563,0.0109421,0.00390788,0.00859733,0.00703418,0.00996508,0,0,0.00762036,0.0273551,0.00410327,0.00664339,0.0212979,0.019344,0,0.00605721,0.00996508,0.00840193,0.0125052,0.00586181,0.0123098,0.0195394,0.00508024,0.00488484,0.0144591,0.00820654,0.00644799,0.00801115,0.0152721,0.00508024,0.0336077,0.00879272,0.0148499,0.0195394,0.023838,0.0220795,0.00429866,0.0543195,0.00449406,0,0,0.0252058,0.0134822,0.00488484,0.00918351,0.0572504,0.0093789,0,0.0912489,0.0636984,0.0113328,0.0945706,0,0.00801115,0.00820654,0.0117236,0.019344,0.00781575,0.080893,0.00664339,0.0205163],[0.00729133,0.00836359,0.00600463,0.00643353,0.00836359,0.0298087,0.00428902,0.00643353,0.00900694,0.00579018,0.0107225,0.00664798,0.0139393,0.00643353,0.00471792,0.0102936,0.0291653,0.00664798,0.00557573,0.0199439,0.015226,0.00750578,0.00814914,0.00729133,0.00857804,0.017585,0.00857804,0.013296,0.0165127,0.00750578,0.00772024,0.0115804,0.00857804,0.0111515,0.00600463,0.00898976,0.00686243,0.0368856,0.0160838,0.0223029,0.03517,0.0379578,0.0167272,0.00557573,0.0538272,0.00750578,0.00471792,0.00450347,0.0193006,0.0102936,0.00750578,0.0128671,0.0450347,0.0139393,0.00450347,0.0849226,0.0488948,0.00814914,0.103794,0.00493237,0.00729133,0.00943584,0.0117948,0.0160838,0.00857804,0.0962885,0.00600463,0.0246619],["mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","MHVV_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change"],[18.7322477517819,17.0663554765358,16.8035332734906,15.1095899140907,14.4180907959381,11.929705086099,11.7625238399448,11.0931168425421,11.0658003717134,10.2005637130452,9.98386570296106,9.43964933709187,9.35706958025152,9.18051209833482,9.12836589005324,9.08492655630683,9.02864019914076,8.88436788317654,8.82741811386132,8.5759054146882,8.56633390253514,8.36083658194085,8.15374383063734,8.10045629535352,7.88950622752983,7.69707136764288,7.60261085282885,7.31107099879663,7.30268217795999,7.21326764173743,7.01289079095987,6.90660944354254,6.88538407375111,6.84122315383581,6.76243165690475,6.65966610899512,6.63435546883538,6.57180579955321,6.34986756860941,6.23434620609876,6.19263576912141,6.15923472909389,6.10562437227988,5.95850229476678,5.92070551691338,5.90175837820986,5.73782514328348,5.7378199477292,5.73780089738143,5.61830652055646,5.57389105462723,5.54654117866497,5.51923294703862,5.4730079702711,5.46458619686597,5.43354772463396,5.36034506736913,5.30017891458485,5.28734801626298,5.23888110583756,5.23154211920185,5.21620756604605,5.21620544646794,5.20229672092416,5.16403514089466,5.16276606240621,5.12304671561778,5.08918615354048]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>cell<\/th>\n      <th>time<\/th>\n      <th>MHVS<\/th>\n      <th>MHVV<\/th>\n      <th>mock<\/th>\n      <th>ns2<\/th>\n      <th>nsp15<\/th>\n      <th>viral_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,9,10,12]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>change_nsp15RNaseL_28S_graph &lt;- change_nsp15RNaseL_28S %&gt;%
  ggplot(aes(x = start, y = fold_change, fill = viral_comp, width=10)) +
  geom_bar(stat = &quot;identity&quot;, position = &#39;identity&#39;) + 
  scale_fill_brewer(palette=&quot;Set1&quot;) +
  facet_grid(viral_comp ~ cell) +
  theme_cowplot() +
  labs(x=&quot;Position&quot;, y=&quot;fold_change&quot;) +
  theme(axis.text.x = element_text(angle = 60, hjust = 1))
change_nsp15RNaseL_28S_graph</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-16-4.png" width="672" /></p>
<pre class="r"><code>#5s

change_nsp15RNaseL_5S &lt;- change_nsp15_5S %&gt;% 
  mutate(MHVS_change = MHVS/nsp15) %&gt;% 
  mutate(MHVV_change = MHVV/nsp15) %&gt;% 
  mutate(mock_change = mock/nsp15) %&gt;%
  #mutate(ns2_change = ns2/nsp15) %&gt;% 
  gather(viral_comp, fold_change, MHVS_change:mock_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  filter(cell == &quot;RNaseL&quot;) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_nsp15RNaseL_5S)</code></pre>
<div id="htmlwidget-dde7bff9c281b6df567f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-dde7bff9c281b6df567f">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11"],["r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240","r5s|NC_000074.6:c123586360-123586240"],[72,74,83,113,75,73,71,76,51,52,77],[73,75,84,114,76,74,72,77,52,53,78],["RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL","RNaseL"],["12","12","12","12","12","12","12","12","12","12","12"],[0.0210738,0.0153083,0,0.0101393,0.0288274,0.00695833,0.00477143,0.0121274,0,0.02505,0.0337976],[0.0442717,0.0350934,0,0.0237555,0.0718065,0.0183565,0,0.0188964,0.0194363,0.0939423,0.0361732],[0.308851,0.233791,0.0430669,0.0824423,0.394985,0.104591,0.0602936,0.0898251,0.0381449,0.155041,0.195647],[0.0265736,0.0201256,0,0.00703418,0.0398603,0.0148499,0.00586181,0.0168039,0.00683878,0.03224,0.0369294],[0.0330255,0.0255197,0.00493237,0.0100792,0.0484659,0.0128671,0.00814914,0.0145827,0.00643353,0.030452,0.0390301],["mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change","mock_change"],[9.35189474799776,9.16119703601531,8.73148202588208,8.17944876577506,8.14975064942568,8.12856043708373,7.39876845900304,6.15970293567035,5.92907781575589,5.09132405096545,5.01272095126582]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>cell<\/th>\n      <th>time<\/th>\n      <th>MHVS<\/th>\n      <th>MHVV<\/th>\n      <th>mock<\/th>\n      <th>ns2<\/th>\n      <th>nsp15<\/th>\n      <th>viral_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,9,10,12]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>#5.8s

change_nsp15RNaseL_5.8S &lt;- change_nsp15_5.8S %&gt;% 
  mutate(MHVS_change = MHVS/nsp15) %&gt;% 
  mutate(MHVV_change = MHVV/nsp15) %&gt;% 
  mutate(mock_change = mock/nsp15) %&gt;%
  #mutate(ns2_change = ns2/nsp15) %&gt;% 
  gather(viral_comp, fold_change, MHVS_change:mock_change) %&gt;%
  filter(fold_change &gt; 0 &amp; fold_change != Inf) %&gt;%
  filter(fold_change &gt;= 5) %&gt;%
  filter(cell == &quot;RNaseL&quot;) %&gt;%
  group_by(start) %&gt;%
  unique() %&gt;%
  arrange(desc(fold_change))

datatable(change_nsp15RNaseL_5.8S)</code></pre>
<div id="htmlwidget-9e9d6dc51c6b35605ae9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9e9d6dc51c6b35605ae9">{"x":{"filter":"none","data":[[],[],[],[],[],[],[],[],[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>cell<\/th>\n      <th>time<\/th>\n      <th>MHVS<\/th>\n      <th>MHVV<\/th>\n      <th>mock<\/th>\n      <th>ns2<\/th>\n      <th>nsp15<\/th>\n      <th>viral_comp<\/th>\n      <th>fold_change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,6,7,8,9,10,12]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="calcualting-dinucelotide-frequencies-for-all-cell-types-by-type-of-viral-infection" class="section level1">
<h1>Calcualting dinucelotide frequencies for all cell types by type of viral infection</h1>
<p>Performing a dinucleotide frequency analysis will help us to confirm RNaseL signatures as a positive control and investigate a potential sequence preference for nsp15 cleavage.</p>
<pre class="r"><code>dinuc &lt;- c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;, &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)
df_dinuc &lt;- data_frame(dinuc)

#18S dinuc frequency 

frequencies_18S &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;,      &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df &lt;- aggregate(cbind(count) ~ dinuc, data = df, sum)
  df &lt;- left_join(df_dinuc, df) %&gt;% 
    replace_na(list(count = 0)) %&gt;% 
    mutate(freq = count/sum(count)) 
  df$name &lt;- basename(x)
  df %&gt;% mutate(name = str_replace(name, &quot;.r18s.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;))
}

freq_18S &lt;- purrr::map_df(data_files_18S, frequencies_18S)

#28S dinuc frequency 

frequencies_28S &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;,      &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df &lt;- aggregate(cbind(count) ~ dinuc, data = df, sum)
  df &lt;- left_join(df_dinuc, df) %&gt;% 
    replace_na(list(count = 0)) %&gt;% 
    mutate(freq = count/sum(count)) 
  df$name &lt;- basename(x)
  df %&gt;% mutate(name = str_replace(name, &quot;.r28s.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;))
}

freq_28S &lt;- purrr::map_df(data_files_28S, frequencies_28S)

# 5S dinuc frequency 

frequencies_5S &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;,      &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df &lt;- aggregate(cbind(count) ~ dinuc, data = df, sum)
  df &lt;- left_join(df_dinuc, df) %&gt;% 
    replace_na(list(count = 0)) %&gt;% 
    mutate(freq = count/sum(count)) 
  df$name &lt;- basename(x)
  df %&gt;% mutate(name = str_replace(name, &quot;.r5s.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;))
}

freq_5S &lt;- purrr::map_df(data_files_5S, frequencies_5S)</code></pre>
</div>
<div id="s-8" class="section level1">
<h1>5.8s</h1>
<pre class="r"><code>frequencies_5.8S &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;,      &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df &lt;- aggregate(cbind(count) ~ dinuc, data = df, sum)
  df &lt;- left_join(df_dinuc, df) %&gt;% 
    replace_na(list(count = 0)) %&gt;% 
    mutate(freq = count/sum(count)) 
  df$name &lt;- basename(x)
  df %&gt;% mutate(name = str_replace(name, &quot;.r5.8s.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;))
}

freq_5.8S &lt;- purrr::map_df(data_files_5.8S, frequencies_5.8S)</code></pre>
<div id="graphing-dinucleotide-frequencies-for-all-cell-types-by-infection-status" class="section level2">
<h2>Graphing dinucleotide frequencies for all cell types by infection status</h2>
<pre class="r"><code>#18S dinuc freq plots 

freq_18S_plot &lt;- freq_18S %&gt;%
  ggplot(aes(x = dinuc,y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() +
  facet_grid(virus ~ cell) +  
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) +
  ggtitle(&quot;18S dinucleotide frequency plots &quot;) 
freq_18S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-19-1.png" width="672" /></p>
<pre class="r"><code>#28S dinuc freq plots 

freq_28S_plot &lt;- freq_28S %&gt;%
  ggplot(aes(x = dinuc,y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) +
  ggtitle(&quot;28S dinucleotide frequency plots &quot;) 
freq_28S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-19-2.png" width="672" /></p>
<pre class="r"><code>#5S dinuc freq plots 

freq_5S_plot &lt;- freq_5S %&gt;%
  ggplot(aes(x = dinuc,y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) +
  ggtitle(&quot;5S dinucleotide frequency plots &quot;) 
freq_5S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-19-3.png" width="672" /></p>
<pre class="r"><code>#5.8S dinuc freq plots 

freq_5.8S_plot &lt;- freq_5.8S %&gt;%
  ggplot(aes(x = dinuc,y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) +
  ggtitle(&quot;5.8S dinucleotide frequency plots &quot;) 
freq_5.8S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-19-4.png" width="672" /></p>
<div id="discussion-of-dinucleotide-analysis" class="section level3">
<h3>Discussion of dinucleotide analysis:</h3>
<p>The relative frequencies and patterns of cleavage in the mock samples are very similar to those observed in the treated samples across all rRNAs. There may be some small differences in frequenct for robust RNase L cleavage sites preferences (UA, UU) between B6 and IFNAR/RNase L KO cells, but overall they are not robust or particulally clear.</p>
</div>
</div>
<div id="rnase-l-substractive-dinucleotide-analysis-of-for-all-cell-types-by-viral-infection" class="section level2">
<h2>RNase L substractive dinucleotide analysis of for all cell types by viral infection</h2>
<p>This is similar to the previous coverage substractive analysis. These plots display dinucleotde cleavage frequencies dependent on RNase L by removing the frequencies detected in the abscence of RNase L.</p>
<pre class="r"><code>#18S

subtractive_18S &lt;- freq_18S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(cell, freq) %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(dinuc:time, B6_RNaseL:IFNAR_RNaseL) %&gt;% 
  gather(cell, freq, B6_RNaseL:IFNAR_RNaseL)

sub18S_freq_plot &lt;- subtractive_18S %&gt;% 
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;,      &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  scale_y_continuous(limits = c(0.00, 0.04)) + 
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) +
  ggtitle(&quot;18S RNase L subtractive dinucleotide frequency plots &quot;) 
sub18S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-20-1.png" width="672" /></p>
<pre class="r"><code>#28S

subtractive_28S &lt;- freq_28S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(cell, freq) %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(dinuc:time, B6_RNaseL:IFNAR_RNaseL) %&gt;% 
  gather(cell, freq, B6_RNaseL:IFNAR_RNaseL)

sub28S_freq_plot &lt;- subtractive_28S %&gt;%
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;,      &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) + 
  scale_y_continuous(limits = c(-0.0, 0.06)) +
  ggtitle(&quot;28S RNase L subtractive dinucleotide frequency plots &quot;) 
sub28S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-20-2.png" width="672" /></p>
<pre class="r"><code>#5S

subtractive_5S &lt;- freq_5S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(cell, freq) %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(dinuc:time, B6_RNaseL:IFNAR_RNaseL) %&gt;% 
  gather(cell, freq, B6_RNaseL:IFNAR_RNaseL)

sub5S_freq_plot &lt;- subtractive_5S %&gt;%
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;, &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;,      &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) + 
    scale_y_continuous(limits = c(-0.0, 0.06)) +
    ggtitle(&quot;5S RNase L subtractive dinucleotide frequency plots &quot;) 
sub5S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-20-3.png" width="672" /></p>
<pre class="r"><code>#5.8S

subtractive_5.8S &lt;- freq_5.8S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(cell, freq) %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(dinuc:time, B6_RNaseL:IFNAR_RNaseL) %&gt;% 
  gather(cell, freq, B6_RNaseL:IFNAR_RNaseL)

sub5.8S_freq_plot &lt;- subtractive_5.8S %&gt;%
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;, &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;,      &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) + 
    scale_y_continuous(limits = c(-0.0, 0.06)) +
    ggtitle(&quot;5S RNase L subtractive dinucleotide frequency plots &quot;) 
sub5.8S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-20-4.png" width="672" /></p>
<div id="discussion-of-rnase-l-subtractive-dinucleotide-analysis" class="section level3">
<h3>Discussion of RNase L subtractive dinucleotide analysis</h3>
<p>The subtractive analysis does faciliate some additional discrmination between “background” frquencies evident in the mock samples from patterns that appear more specific to the treated samples. Although overall the patterns aren’t very consistent, cleavage at dinucleotides where the 3´-position is U or C have the greatest frequencies of cleavage, especially in the B6 cells. In the 28s rRNA, a preference for UA, UG, UC, and UU is more pronounced.</p>
</div>
</div>
<div id="nsp15-subtractive-dinucleotide-analysis-of-for-all-cell-types-by-viral-infection" class="section level2">
<h2>Nsp15 subtractive dinucleotide analysis of for all cell types by viral infection</h2>
<pre class="r"><code>#18S

endoU_subtractive_18S &lt;- freq_18S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(virus, freq) %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(dinuc:time, MHVS_nsp15:mock_nsp15) %&gt;% 
  gather(virus, freq, MHVS_nsp15:mock_nsp15)

endoU_sub18S_freq_plot &lt;- endoU_subtractive_18S %&gt;% 
  ggplot(aes(x = dinuc , y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;,      &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  scale_y_continuous(limits = c(0.00, 0.06)) +
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) +
  ggtitle(&quot;18S nsp15 subtractive dinucleotide frequency plots &quot;) 
endoU_sub18S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-21-1.png" width="672" /></p>
<pre class="r"><code>#28S

endoU_subtractive_28S &lt;- 
  freq_28S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(virus, freq) %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(dinuc:time, MHVS_nsp15:mock_nsp15) %&gt;% 
  gather(virus, freq, MHVS_nsp15:mock_nsp15)

endoU_sub28S_freq_plot &lt;- endoU_subtractive_28S %&gt;% 
  ggplot(aes(x = dinuc , y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  scale_y_continuous(limits = c(0.0, 0.06)) +
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) +
  ggtitle(&quot;28S nsp15 subtractive dinucleotide frequency plots &quot;) 
endoU_sub28S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-21-2.png" width="672" /></p>
<pre class="r"><code>#5S

endoU_subtractive_5S &lt;- freq_5S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(virus, freq) %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(dinuc:time, MHVS_nsp15:mock_nsp15) %&gt;% 
  gather(virus, freq, MHVS_nsp15:mock_nsp15)

endoU_sub5S_freq_plot &lt;- endoU_subtractive_5S %&gt;% 
  ggplot(aes(x = dinuc , y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  scale_y_continuous(limits = c(0.0, 0.06)) +
  facet_grid(cell ~ virus) + 
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) +
  ggtitle(&quot;5S nsp15 subtractive dinucleotide frequency plots &quot;) 
endoU_sub5S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-21-3.png" width="672" /></p>
<pre class="r"><code>#5.8S

endoU_subtractive_5.8S &lt;- freq_5.8S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(virus, freq) %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(dinuc:time, MHVS_nsp15:mock_nsp15) %&gt;% 
  gather(virus, freq, MHVS_nsp15:mock_nsp15)

endoU_sub5.8S_freq_plot &lt;- endoU_subtractive_5.8S %&gt;% 
  ggplot(aes(x = dinuc , y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  scale_y_continuous(limits = c(0.0, 0.06)) +
  facet_grid(cell ~ virus) + 
  theme(axis.text.x = element_text(angle = 60, hjust = 1, size = 8)) +
  ggtitle(&quot;5S nsp15 subtractive dinucleotide frequency plots &quot;) 
endoU_sub5.8S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_final_files/figure-html/unnamed-chunk-21-4.png" width="672" /></p>
<div id="discussion-of-nsp15-subtractive-analysis" class="section level3">
<h3>Discussion of nsp15 subtractive analysis:</h3>
<p>For each rRNA, there were no obvious differences in dincuelotide cleavage preferecnce between the mock and infected samples. Whereas the RNase L subtractive analysis exposed some differences in dincleotide cleavage frequency between the mock and infected samples, which suggests that there is no nsp15 dependent cleavage preference in rRNA or that the cleavage freuqncy is less robust than RNase L and cannot be distinguished from the background.</p>
</div>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
