<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Rachel Ancar" />

<meta name="date" content="2018-04-25" />

<title>Analysis of rRNA to Detect EndoU Cleavage Sites</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">EndoU Collaboration Data Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="MHV_endoU_analysis_final.html">MHV RNA analysis</a>
</li>
<li>
  <a href="MHV_endo_analysis_reseq_final.html">MHV RNA re-seq analysis</a>
</li>
<li>
  <a href="rRNA_endoU_analysis.html">rRNA analysis</a>
</li>
<li>
  <a href="mRNA_endoU_analysis.html">mRNA analysis</a>
</li>
<li>
  <a href="U6_23-Cyclic_Phosphate_Analysis.html">U6 RNA analysis</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">

<div class="btn-group pull-right">
<button type="button" class="btn btn-default btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Analysis of rRNA to Detect EndoU Cleavage Sites</h1>
<h4 class="author"><em>Rachel Ancar</em></h4>
<h4 class="date"><em>04/25/2018</em></h4>

</div>


<p>Prior to this analysis, 2´3´-cyclic phosphate RNA-sequencing libraries were prepared from mouse macrophages: B6 (WT), IFNAR KO, RNaseL KO. These cells were either mock infected or infected with WT MHV virus (from Volker or Susan), MHV mutated to inactivate ns2 (phosphodiesterase) activity, or MHV mutated to inactivate nsp15 (endoribonuclease) activity for 9 and 12 hours.</p>
<p>These libraries were processed to generate bedgraph files containing sites of 3´-cleavage and the associated dinucleotide for 18S, 28S, and 5s rRNA.</p>
<pre class="r"><code>library(tidyverse)
library(readr)
library(dplyr)
library(ggplot2)
library(valr)
library(stringr)
library(cowplot)
library(RColorBrewer)
library(purrr)
library(shiny)
library(DT)</code></pre>
<div id="coverage-plots-for-18s-28s-and-5s-rrna-by-cell-type-and-type-of-viral-infection" class="section level2">
<h2>Coverage plots for 18S, 28S, and 5s rRNA by cell type and type of viral infection</h2>
<p>These plots display coverage across the entire 18S, 28S, and 5s rRNA sequence. Counts are normalized by library size.</p>
<pre class="r"><code>sample_info_table &lt;- read_tsv(&quot;sample_info.txt&quot;, col_names = c(&quot;cell type&quot;, &quot;virus type&quot;, &quot;time&quot;))
datatable(sample_info_table)</code></pre>
<div id="htmlwidget-2fc697b6a55cbde92786" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2fc697b6a55cbde92786">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36"],["B6","B6","B6","B6","B6","IFNAR","IFNAR","IFNAR","IFNAR","IFNAR","RnaseL","RnaseL","RnaseL","RnaseL","RnaseL","B6","B6","B6","B6","B6","IFNAR","IFNAR","IFNAR","IFNAR","IFNAR","RnaseL","RnaseL","RnaseL","RnaseL","RnaseL","B6","IFNAR","RNaseL","B6","IFNAR","RNaseL"],["Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","Mock","MHV wt Volker","MHV wt susan","NS2susan mut 3","nsp15HA Eveline P1","mock","mock","mock","mock","mock","mock"],["12","12","12","12","12","12","12","12","12","12","12","12","12","12","12","9","9","9","9","9","9","9","9","9","9","9","9","9","9","9","0/12","0/12","0/12","0/9","0/9","0/9"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>cell type<\/th>\n      <th>virus type<\/th>\n      <th>time<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}]}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>#18S rRNA coverage plots

data_dir_18S = &quot;~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/rRNA_bg/mm18S/neg/&quot;
data_files_18S = list.files(data_dir_18S, full.names = T)

read_file &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df$name &lt;- basename(x)
  df
}

table_18S &lt;- purrr::map_df(data_files_18S, read_file) %&gt;%
  mutate(name = str_replace(name, &quot;.umi.18S.dinuc.neg.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;), sep = &#39;_&#39;)

plot_18S &lt;- ggplot(table_18S, aes(x = start, y = normalized_count)) +
    #geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) 
    geom_line(aes(color = time)) + 
    scale_color_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    ggtitle(&quot;18S rRNA coverage plots&quot;)
plot_18S</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-2-2.png" width="672" /></p>
<pre class="r"><code># Previously documented RNaseL cleavage site 542 in 18S rRNA

plot_18S_542 &lt;- ggplot(table_18S, aes(x = start, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    #geom_line(aes(color = time)) + 
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    xlim(530, 560) +
    ggtitle(&quot;RNaseL cleavage site 542 in 18S rRNA&quot;)
plot_18S_542</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-2-3.png" width="672" /></p>
<pre class="r"><code>#28S rRNA coverage plots

data_files_28S = list.files(&quot;~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/rRNA_bg/mm28S/neg/&quot; , full.names = T)

read_file &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df$name &lt;- basename(x)
  df
}

table_28S &lt;- purrr::map_df(data_files_28S, read_file) %&gt;%
  mutate(name = str_replace(name, &quot;.umi.28S.dinuc.neg.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;), sep = &#39;_&#39;)

plot_28S &lt;- ggplot(table_28S, aes(x = end, y = normalized_count)) +
    #geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    geom_line(aes(color = time)) + 
    scale_colour_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    ggtitle(&quot;28S rRNA coverage plots&quot;)
plot_28S</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-2-4.png" width="672" /></p>
<pre class="r"><code>#5S rRNA

data_dir_5S = &quot;~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/rRNA_bg/mm5S/neg&quot;
data_files_5S = list.files(data_dir_5S, full.names = T)

read_file &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df$name &lt;- basename(x)
  df
}

table_5S &lt;- purrr::map_df(data_files_5S, read_file) %&gt;%
  mutate(name = str_replace(name, &quot;.umi.5S.dinuc.neg.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;), sep = &#39;_&#39;)

plot_5S &lt;- ggplot(table_5S, aes(x = end, y = normalized_count)) +
    #geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    geom_line(aes(color = time)) + 
    scale_colour_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    ggtitle(&quot;5S rRNA coverage plots&quot;)
plot_5S</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-2-5.png" width="672" /></p>
<div id="disscusion-of-ribosomal-rna-coverage-plots" class="section level3">
<h3>Disscusion of ribosomal RNA coverage plots</h3>
<p>First, it is clear that there is signfiicant non-specific cleavage of ribosomal RNA that has been captured in these libaries. There is robust cleavage in the mock samples of all analyzed rRNAs. Across all rRNAs and conditions, most cleavage is detected at 9 vs. 12 hpi. The exception to this observation is for rRNA cleavage from B6 cells infected with MHV ns2 mutant. In these cells, there is more clevage at 12 hpi, which may be a reflection of specific increased RNase L activation. The 5s rRNA signal does not follow these patterns observed in the 18s and 28 rRNA. The 5s rRNA signal in general is much lower and there do not seem to be any speciifc changes in signal captured across cell type or infection.</p>
<p>Speciifc cleavage can still be detected at the known RNase L cleavage site in 18s rRNA at nucleotide 542. At this site, there is very little cleavage in the mock samples, and clear induction of cleavage at this site in B6 cells infected with the nsp15 mutant relative to the other virus types.</p>
<p>Overall, it is difficult to determine if there are any specific sites of clevaage that change in the abscence of nsp15 activity due to the signfiicant background signal.</p>
</div>
</div>
<div id="rnase-l-substractive-coverage-analysis-of-rrna-reads-for-all-cell-types-by-type-of-viral-infection" class="section level2">
<h2>RNase L substractive coverage analysis of rRNA reads for all cell types by type of viral infection</h2>
<p>The normalized counts detected in RNase L KO cells for each type of viral infection were subtracted from the normalized counts detected in WT B6 and IFNAR KO cells. This substractive analysis will emphasize RNase L dependent cleavage events by reporting signals that occur only in the presence of RNase L.</p>
<pre class="r"><code>#18S rRNA RNase L substractive analysis 

subtractive_18S &lt;- table_18S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count) 
subtractive_18S[is.na(subtractive_18S)] &lt;- 0

subtractive_18S &lt;- subtractive_18S %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;%                     
  dplyr::select(start:end, virus:time, B6_RNaseL:IFNAR_RNaseL) %&gt;%       gather(cell, normalized_count, B6_RNaseL:IFNAR_RNaseL)
                                      
#18s rRNA RNase L dependent cleavage plots

sub18S_plot &lt;- ggplot(subtractive_18S, aes(x = end, y = normalized_count)) +
    #geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    geom_line(aes(color = time)) + 
    scale_color_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) + 
    scale_y_continuous(limits = c(0 , 0.02)) +
    ggtitle(&quot;18s rRNA RNase L dependent cleavage plots&quot;)
sub18S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-3-1.png" width="672" /></p>
<pre class="r"><code>#28S rRNA RNase L substractive analysis 

subtractive_28S &lt;- table_28S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count) 
subtractive_28S[is.na(subtractive_28S)] &lt;- 0

subtractive_28S &lt;- subtractive_28S %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(start:end, virus:time, B6_RNaseL:IFNAR_RNaseL) %&gt;%      gather(cell, normalized_count, B6_RNaseL:IFNAR_RNaseL)

#28s rRNA RNase L dependent cleavage plots
                                      
sub28S_plot &lt;- ggplot(subtractive_28S, aes(x = end, y = normalized_count)) +
    #geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    geom_line(aes(color = time)) + 
    scale_color_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +                  scale_y_continuous(limits = c(0 , 0.02)) +
   ggtitle(&quot;28s rRNA RNase L dependent cleavage plots&quot;)
sub28S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-3-2.png" width="672" /></p>
<pre class="r"><code>#5S rRNA RNase L substractive analysis 

subtractive_5S &lt;- table_5S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(cell, normalized_count) 
subtractive_5S[is.na(subtractive_5S)] &lt;- 0

subtractive_5S &lt;- subtractive_5S %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(start:end, virus:time, B6_RNaseL:IFNAR_RNaseL) %&gt;%    
  gather(cell, normalized_count, B6_RNaseL:IFNAR_RNaseL)

#5s rRNA RNase L dependent cleavage plots
                                      
sub5S_plot &lt;- ggplot(subtractive_5S, aes(x = end, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    #geom_line(aes(color = time)) + 
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +              
    scale_y_continuous(limits = c(0 , 0.005)) +
    ggtitle(&quot;5s rRNA RNase L dependent cleavage plots&quot;)
sub5S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-3-3.png" width="672" /></p>
<div id="discussion-of-rnase-l-subtractive-analysis" class="section level3">
<h3>Discussion of RNase L subtractive analysis:</h3>
<p>In the 18s and 28 rRNA, there is still significant signal in the mock samples, which seems to occur generally across the rRNA sequence without a preference for time. However, the signal in the treated samples do have a more pronounced pattern preference across the RNA, and the RNase L signal occurs most robustly at 12 hpi, except in the nsp15 mutant viral infection, where the signal is most enriched at 9 hpi. In the 5s rRNA analysis, there aren’t any clear or consistent cleavage patterns between cell type of viral infection type.</p>
</div>
</div>
<div id="endou-mutant-nsp15-substractive-coverage-analysis-of-mhv-strand-for-all-cell-types-by-type-of-viral-infection" class="section level2">
<h2>EndoU mutant (nsp15) substractive coverage analysis of MHV (+) strand for all cell types by type of viral infection</h2>
<p>This is very similar to the analysis above, expect it will substract signal occuring in the abscence of nsp15 actviity, which will emphasize sites specific to nsp15 cleavage actviity.</p>
<pre class="r"><code>#18S rRNA EndoU substractive analysis 

subtractive_nsp15_18S &lt;- table_18S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_18S[is.na(subtractive_nsp15_18S)] &lt;- 0

subtractive_nsp15_18S &lt;- subtractive_nsp15_18S %&gt;%       
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;%
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;%
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;%
  mutate(mock_nsp15 = mock - nsp15) %&gt;%
  dplyr::select(start:end, cell:time, MHVS_nsp15:mock_nsp15) %&gt;% 
  gather(virus, normalized_count, MHVS_nsp15:mock_nsp15)
                                      
subnsp15_18S_plot &lt;- ggplot(subtractive_nsp15_18S, aes(x = end, y = normalized_count)) +
    #geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
    geom_line(aes(color = time)) + 
    scale_color_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +  
    scale_y_continuous(limits = c(0.0, 0.02)) +
    ggtitle(&quot;18S rRNA EndoU substractive analysis &quot;)
subnsp15_18S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-4-1.png" width="672" /></p>
<pre class="r"><code>#28S

subtractive_nsp15_28S &lt;- table_28S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_28S[is.na(subtractive_nsp15_28S)] &lt;- 0

subtractive_nsp15_28S &lt;- subtractive_nsp15_28S %&gt;%   
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(start:end, cell:time, MHVS_nsp15:mock_nsp15) %&gt;%    gather(virus, normalized_count, MHVS_nsp15:mock_nsp15)
                                      
subnsp15_28S_plot &lt;- ggplot(subtractive_nsp15_28S, aes(x = end, y = normalized_count)) +
    #geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    geom_line(aes(color = time)) + 
    scale_color_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +     
    scale_y_continuous(limits = c(0.0, 0.02)) +
    ggtitle(&quot;28S rRNA EndoU substractive analysis &quot;)
subnsp15_28S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-4-2.png" width="672" /></p>
<pre class="r"><code>#5S

subtractive_nsp15_5S &lt;- table_5S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_5S[is.na(subtractive_nsp15_5S)] &lt;- 0

subtractive_nsp15_5S &lt;- subtractive_nsp15_5S %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(start:end, cell:time, MHVS_nsp15:mock_nsp15) %&gt;%    gather(virus, normalized_count, MHVS_nsp15:mock_nsp15)
                                      
subnsp15_5S_plot &lt;- ggplot(subtractive_nsp15_5S, aes(x = end, y = normalized_count)) +
    #geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    geom_line(aes(color = time)) + 
    scale_color_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +                   scale_y_continuous(limits = c(0.0, 0.02)) +
    ggtitle(&quot;5S rRNA EndoU substractive analysis &quot;)
subnsp15_5S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-4-3.png" width="672" /></p>
<div id="discussion-of-nsp15-substractive-analysis" class="section level3">
<h3>Discussion of nsp15 substractive analysis:</h3>
<p>In the 18 and 28 rRNA, there is significant signal in the mock samples across the entire rRNA. In the treated samples, the B6 WT cells the signal occurs at specific regions of the RNA and occurs most robustly at 12 hpi. There is very little cleavage in the IFNAR KO cells that is nsp15 dependent. However, there is signal dectected across the entire RNA in the two MHV WT virus infections in RNase L KO cells, but this may be non-specific because the pattern is similar to that observed in the mock. While these data are noisy, in general there does seem to be a pattern suggesting some type of interaction between nsp15 activity and IFN signalling, and perhaps a type of competition when there is active RNAse L. There is almost no nsp15 dependent signal in the 5s rRNA.</p>
</div>
</div>
<div id="baseline-subtractive-analysis-remvoving-reads-that-occur-in-the-mock-infected-samples" class="section level2">
<h2>Baseline subtractive analysis: remvoving reads that occur in the mock infected samples</h2>
<p>This analysis attempts to establish a signal “baseline” by determining the signal at each base in the RNA that remains after subtracting the signal that occured in the mock samples at the same base.</p>
<pre class="r"><code>#18S mock subtractive analysis 

subtractive_mock_18S &lt;- table_18S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_mock_18S[is.na(subtractive_mock_18S)] &lt;- 0

subtractive_mock_18S &lt;- subtractive_mock_18S %&gt;% 
  mutate(MHVSbase = MHVS - mock) %&gt;% 
  mutate(MHVVbase = MHVV - mock) %&gt;% 
  mutate(ns2base = ns2 - mock) %&gt;% 
  mutate(nsp15base = nsp15 - mock) %&gt;% 
  dplyr::select(chrom:end, cell:time, MHVSbase:nsp15base) %&gt;% 
  gather(virus, normalized_count, MHVSbase:nsp15base)
                                      
submock_18S_plot &lt;- ggplot(subtractive_mock_18S, aes(x = end, y = normalized_count)) +
    geom_line(aes(color = time)) + 
    scale_colour_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    scale_y_continuous(limits = c(0, 0.03)) + 
    labs(x=&quot;Position&quot;, y=&quot;Normalized counts (counts/total reads)&quot;) +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    ggtitle(&quot;18S rRNA baseline coverage plots &quot;) 
submock_18S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-5-1.png" width="672" /></p>
<pre class="r"><code>#28S mock subtractive analysis 

subtractive_mock_28S &lt;- table_28S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_mock_28S[is.na(subtractive_mock_28S)] &lt;- 0

subtractive_mock_28S &lt;- subtractive_mock_28S %&gt;% 
  mutate(MHVSbase = MHVS - mock) %&gt;% 
  mutate(MHVVbase = MHVV - mock) %&gt;% 
  mutate(ns2base = ns2 - mock) %&gt;% 
  mutate(nsp15base = nsp15 - mock) %&gt;% 
  dplyr::select(start:end, cell:time, MHVSbase:nsp15base) %&gt;%     gather(virus, normalized_count, MHVSbase:nsp15base)
                                      
submock_28S_plot &lt;- ggplot(subtractive_mock_28S, aes(x = end, y = normalized_count)) +
    geom_line(aes(color = time)) + 
    scale_color_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    scale_y_continuous(limits = c(0, 0.03)) + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    ggtitle(&quot;28S rRNA baseline coverage plots&quot;) 
submock_28S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-5-2.png" width="672" /></p>
<pre class="r"><code>#5S mock subtractive analysis 

subtractive_mock_5S &lt;- 
  table_5S %&gt;% 
  dplyr::select(-count, -dinuc) %&gt;% 
  spread(virus, normalized_count) 
subtractive_mock_5S[is.na(subtractive_mock_5S)] &lt;- 0

subtractive_mock_5S &lt;- subtractive_mock_5S %&gt;% 
  mutate(MHVSbase = MHVS - mock) %&gt;% 
  mutate(MHVVbase = MHVV - mock) %&gt;% 
  mutate(ns2base = ns2 - mock) %&gt;% 
  mutate(nsp15base = nsp15 - mock) %&gt;% 
  dplyr::select(start:end, cell:time, MHVSbase:nsp15base) %&gt;% 
  gather(virus, normalized_count, MHVSbase:nsp15base)
                                      
submock_5S_plot &lt;- ggplot(subtractive_mock_5S, aes(x = end, y = normalized_count, fill = time)) +
    geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    #geom_line(aes(color = time)) + 
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    scale_y_continuous(limits = c(0, 0.01)) + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    ggtitle(&quot;5S rRNA baseline coverage plots&quot;) 
submock_5S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-5-3.png" width="672" /></p>
<div id="discussion-of-baseline-analysis" class="section level3">
<h3>Discussion of baseline analysis</h3>
<p>Removing the signal that occured in the mock samples reveals a few differenes as compared to the inital coverage graphs. In the 18s and 28s rRNA most of the captured signal is at 12 hpi and the MHVV virus infected samples have the lowest cleavage across all cell types. The pattern of cleavage shifts a bit in the RNase L KO cells as compared to the B6 and IFNAR KO cells, but there aren’t any clear sites of nsp15 dependent cleavage relative to the background.</p>
<p>In the 5s rRNA, after removing signal from the mock samples, there is very little clevage evident in the RNA. There does seem to be a signal around nt 50 that may be RNase L dependent.</p>
</div>
</div>
<div id="endou-mutant-nsp15-substractive-coverage-analysis-of-mhv-strand-for-all-cell-types-by-type-of-viral-infection-from-baseline" class="section level2">
<h2>EndoU mutant (nsp15) substractive coverage analysis of MHV (+) strand for all cell types by type of viral infection from baseline</h2>
<pre class="r"><code>#18S nsp15 subbstractive analysis after removing mock signal for &quot;baseline&quot; values

subtractive_nsp15_baseline18S &lt;- subtractive_mock_18S %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_baseline18S[is.na(subtractive_nsp15_baseline18S)] &lt;- 0

subtractive_nsp15_baseline18S &lt;- subtractive_nsp15_baseline18S %&gt;% 
  mutate(MHVS_nsp15 = MHVSbase - nsp15base) %&gt;% 
  mutate(MHVV_nsp15 = MHVVbase - nsp15base) %&gt;% 
  mutate(ns2_nsp15 = ns2base - nsp15base) %&gt;% 
  dplyr::select(start:end, cell:time, MHVS_nsp15:ns2_nsp15) %&gt;% 
  gather(virus, normalized_count, MHVS_nsp15:ns2_nsp15)
                                      
subnsp15_baseline18S_plot &lt;- 
  ggplot(subtractive_nsp15_baseline18S, aes(x = end, y = normalized_count)) +
    geom_line(aes(color = time)) + 
    scale_colour_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    scale_y_continuous(limits = c(0, 0.02)) + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    ggtitle(&quot;Baseline 18S rRNA nsp15 dependent analysis&quot;) 
subnsp15_baseline18S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-6-1.png" width="672" /></p>
<pre class="r"><code>#28S

subtractive_nsp15_baseline28S &lt;- subtractive_mock_28S %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_baseline28S[is.na(subtractive_nsp15_baseline28S)] &lt;- 0

subtractive_nsp15_baseline28S &lt;- subtractive_nsp15_baseline28S %&gt;% 
  mutate(MHVS_nsp15 = MHVSbase - nsp15base) %&gt;% 
  mutate(MHVV_nsp15 = MHVVbase - nsp15base) %&gt;% 
  mutate(ns2_nsp15 = ns2base - nsp15base) %&gt;% 
  dplyr::select(start:end, cell:time, MHVS_nsp15:ns2_nsp15) %&gt;% 
  gather(virus, normalized_count, MHVS_nsp15:ns2_nsp15)
                                      
subnsp15_baseline28S_plot &lt;- ggplot(subtractive_nsp15_baseline28S, aes(x = end, y = normalized_count)) +
    #geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    geom_line(aes(color = time)) + 
    scale_color_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    scale_y_continuous(limits = c(0, 0.01)) + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    ggtitle(&quot;Baseline 28S rRNA nsp15 dependent analysis&quot;) 
subnsp15_baseline28S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-6-2.png" width="672" /></p>
<pre class="r"><code>#5S

subtractive_nsp15_baseline5S &lt;- subtractive_mock_5S %&gt;% 
  spread(virus, normalized_count) 
subtractive_nsp15_baseline5S[is.na(subtractive_nsp15_baseline5S)] &lt;- 0

subtractive_nsp15_baseline5S &lt;- subtractive_nsp15_baseline5S %&gt;% 
  mutate(MHVS_nsp15 = MHVSbase - nsp15base) %&gt;% 
  mutate(MHVV_nsp15 = MHVVbase - nsp15base) %&gt;% 
  mutate(ns2_nsp15 = ns2base - nsp15base) %&gt;% 
  dplyr::select(start:end, cell:time, MHVS_nsp15:ns2_nsp15) %&gt;% 
  gather(virus, normalized_count, MHVS_nsp15:ns2_nsp15)
                                      
subnsp15_baseline5S_plot &lt;- ggplot(subtractive_nsp15_baseline5S, aes(x = end, y = normalized_count, fill = time)) +
    geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) +
    #geom_line(aes(color = time)) + 
    scale_fill_brewer(palette = &#39;Set1&#39;) + 
    theme_cowplot() + 
    scale_y_continuous(limits = c(0, 0.01)) + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    ggtitle(&quot;Baseline 5S rRNA nsp15 dependent analysis&quot;) 
subnsp15_baseline5S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-6-3.png" width="672" /></p>
<div id="discussion-of-nsp15-dependent-analysis-after-baseline-substraction" class="section level3">
<h3>Discussion of nsp15 dependent analysis after baseline substraction</h3>
<p>The nsp15 substractive analysis at baseline is very similar to the original analysis without removing the mock signal. However, there is a some what clearer peak in the beginning region of the 18s RNA in the RNase L KO cells, but not in the RNase L KO cells infected with ns2 mutant virus. In both the 18s and 28s rRNA, there is almost no nsp15-dependnent signal captured in the RNase L KO cells infected with ns2 mutant virus. But, there is signal in the RNase L substratcive analysis. This suggests that nsp15 cleavage is being inhibited by the mutant nsp15 virus. In the 5s rRNA there is very little singal, expect for this peak around 50 nt that only occurs in the IFNAR KO cells. This signal also occurs in the RNase L dependent analysis, so it is msot likely actually dependnent on another unknown factor.</p>
<p>Overall, there are still significant differences in signal captured between the cell types and types of viral infections, suggesting there are several interactions contributing to these cleavage events.</p>
</div>
</div>
<div id="calcualting-dinucelotide-frequencies-for-all-cell-types-by-type-of-viral-infection" class="section level2">
<h2>Calcualting dinucelotide frequencies for all cell types by type of viral infection</h2>
<p>Performing a dinucleotide frequency analysis will help us to confirm RNaseL signatures as a positive control and investigate a potential sequence preference for nsp15 cleavage.</p>
<pre class="r"><code>dinuc &lt;- c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;, &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)
df_dinuc &lt;- data_frame(dinuc)

#18S dinuc frequency 

frequencies_18S &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df &lt;- aggregate(cbind(count) ~ dinuc, data = df, sum)
  df &lt;- left_join(df_dinuc, df) %&gt;% replace_na(list(count = 0)) %&gt;% 
  mutate(freq = count/sum(count))
  df$name &lt;- basename(x)
  df %&gt;% mutate(name = str_replace(name, &quot;.umi.18S.dinuc.neg.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;))
}

freq_18S &lt;- purrr::map_df(data_files_18S, frequencies_18S)

#28S dinuc frequency 

frequencies_28S &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df &lt;- aggregate(cbind(count) ~ dinuc, data = df, sum)
  df &lt;- left_join(df_dinuc, df) %&gt;% replace_na(list(count = 0)) %&gt;% 
  mutate(freq = count/sum(count))
  df$name &lt;- basename(x)
  df %&gt;% mutate(name = str_replace(name, &quot;.umi.28S.dinuc.neg.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;))
}

freq_28S &lt;- purrr::map_df(data_files_28S, frequencies_28S)

# 5S dinuc frequency 

frequencies_5S &lt;- function(x) {
  df &lt;- readr::read_tsv(x, col_names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;count&quot;, &quot;normalized_count&quot;, &quot;dinuc&quot;))
  df &lt;- aggregate(cbind(count) ~ dinuc, data = df, sum)
  df &lt;- left_join(df_dinuc, df) %&gt;% replace_na(list(count = 0)) %&gt;% 
  mutate(freq = count/sum(count))
  df$name &lt;- basename(x)
  df %&gt;% mutate(name = str_replace(name, &quot;.umi.5S.dinuc.neg.bg&quot;, &quot;&quot;)) %&gt;%
  separate(name, into = c(&#39;cell&#39;, &#39;virus&#39;, &#39;time&#39;))
} 

freq_5S &lt;- purrr::map_df(data_files_5S, frequencies_5S)</code></pre>
</div>
<div id="graphing-dinucleotide-frequencies-for-all-cell-types-by-infection-status" class="section level2">
<h2>Graphing dinucleotide frequencies for all cell types by infection status</h2>
<pre class="r"><code>#18S dinuc freq plots 

freq_18S_plot &lt;- freq_18S %&gt;%
  ggplot(aes(x = dinuc,y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() +
  facet_grid(virus ~ cell) +  
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle(&quot;18S dinucleotide frequency plots &quot;) 
freq_18S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-8-1.png" width="672" /></p>
<pre class="r"><code>#28S dinuc freq plots 

freq_28S_plot &lt;- freq_28S %&gt;%
  ggplot(aes(x = dinuc,y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle(&quot;28S dinucleotide frequency plots &quot;) 
freq_28S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-8-2.png" width="672" /></p>
<pre class="r"><code>#5S dinuc freq plots 

freq_5S_plot &lt;- freq_5S %&gt;%
  ggplot(aes(x = dinuc,y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle(&quot;5S dinucleotide frequency plots &quot;) 
freq_5S_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-8-3.png" width="672" /></p>
<div id="discussion-of-dinucleotide-analysis" class="section level3">
<h3>Discussion of dinucleotide analysis:</h3>
<p>The significant amount of non-specific rRNA cleavage makes it difficult to draw conclusions about the dinucleotide cleavage patterns. There relative frequencies and patterns of cleavage in the mock samples are very similar to that observed in the treated samples across all rRNAs. There may be some small differences in frequenct for robust RNase L cleavage sites preferences (UA, UU) between B6 and IFNAR/RNase L KO cells, but overall they are not robust or particulally clear.</p>
</div>
</div>
<div id="rnase-l-substractive-dinucleotide-analysis-of-for-all-cell-types-by-viral-infection" class="section level2">
<h2>RNase L substractive dinucleotide analysis of for all cell types by viral infection</h2>
<p>This is similar to the previous coverage substractive analysis. These plots display dinucleotde cleavage frequencies dependent on RNase L by removing the frequencies detected in the abscence of RNase L.</p>
<pre class="r"><code>#18S

subtractive_18S &lt;- freq_18S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(cell, freq) %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(dinuc:time, B6_RNaseL:IFNAR_RNaseL) %&gt;% 
  gather(cell, freq, B6_RNaseL:IFNAR_RNaseL)

sub18S_freq_plot &lt;- subtractive_18S %&gt;% 
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;,      &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  scale_y_continuous(limits = c(0.00, 0.04)) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle(&quot;18S RNase L subtractive dinucleotide frequency plots &quot;) 
sub18S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-9-1.png" width="672" /></p>
<pre class="r"><code>#28S

subtractive_28S &lt;- freq_28S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(cell, freq) %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(dinuc:time, B6_RNaseL:IFNAR_RNaseL) %&gt;% 
  gather(cell, freq, B6_RNaseL:IFNAR_RNaseL)

sub28S_freq_plot &lt;- subtractive_28S %&gt;%
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;,      &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
  scale_y_continuous(limits = c(-0.0, 0.06)) +
  ggtitle(&quot;28S RNase L subtractive dinucleotide frequency plots &quot;) 
sub28S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-9-2.png" width="672" /></p>
<pre class="r"><code>#5S

subtractive_5S &lt;- freq_5S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(cell, freq) %&gt;% 
  mutate(B6_RNaseL = B6 - RNaseL) %&gt;% 
  mutate(IFNAR_RNaseL = IFNAR - RNaseL) %&gt;% 
  dplyr::select(dinuc:time, B6_RNaseL:IFNAR_RNaseL) %&gt;% 
  gather(cell, freq, B6_RNaseL:IFNAR_RNaseL)

sub5S_freq_plot &lt;- subtractive_5S %&gt;%
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;, &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;,      &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
    scale_y_continuous(limits = c(-0.0, 0.06)) +
    ggtitle(&quot;5S RNase L subtractive dinucleotide frequency plots &quot;) 
sub5S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-9-3.png" width="672" /></p>
<div id="discussion-of-rnase-l-subtractive-dinucleotide-analysis" class="section level3">
<h3>Discussion of RNase L subtractive dinucleotide analysis</h3>
<p>The subtractive analysis does faciliate some additional discrmination between “background” frquencies evident in the mock samples from patterns that appear more specific to the treated samples. Although overall the patterns aren’t very consistent, cleavage at dinucleotides where the 3´-position is U or C have the greatest frequencies of cleavage, especially in the B6 cells. In the 28s rRNA, a preference for UA, UG, UC, and UU is more pronounced.</p>
</div>
</div>
<div id="nsp15-subtractive-dinucleotide-analysis-of-for-all-cell-types-by-viral-infection" class="section level2">
<h2>Nsp15 subtractive dinucleotide analysis of for all cell types by viral infection</h2>
<pre class="r"><code>#18S

endoU_subtractive_18S &lt;- freq_18S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(virus, freq) %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(dinuc:time, MHVS_nsp15:mock_nsp15) %&gt;% 
  gather(virus, freq, MHVS_nsp15:mock_nsp15)

endoU_sub18S_freq_plot &lt;- endoU_subtractive_18S %&gt;% 
  ggplot(aes(x = dinuc , y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;,      &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  scale_y_continuous(limits = c(0.00, 0.06)) +
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle(&quot;18S nsp15 subtractive dinucleotide frequency plots &quot;) 
endoU_sub18S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-10-1.png" width="672" /></p>
<pre class="r"><code>#28S

endoU_subtractive_28S &lt;- 
  freq_28S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(virus, freq) %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(dinuc:time, MHVS_nsp15:mock_nsp15) %&gt;% 
  gather(virus, freq, MHVS_nsp15:mock_nsp15)

endoU_sub28S_freq_plot &lt;- endoU_subtractive_28S %&gt;% 
  ggplot(aes(x = dinuc , y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  scale_y_continuous(limits = c(0.0, 0.06)) +
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle(&quot;28S nsp15 subtractive dinucleotide frequency plots &quot;) 
endoU_sub28S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-10-2.png" width="672" /></p>
<pre class="r"><code>#5S

endoU_subtractive_5S &lt;- freq_5S %&gt;% 
  dplyr::select(dinuc, freq:time) %&gt;% 
  spread(virus, freq) %&gt;% 
  mutate(MHVS_nsp15 = MHVS - nsp15) %&gt;% 
  mutate(MHVV_nsp15 = MHVV - nsp15) %&gt;% 
  mutate(ns2_nsp15 = ns2 - nsp15) %&gt;% 
  mutate(mock_nsp15 = mock - nsp15) %&gt;% 
  dplyr::select(dinuc:time, MHVS_nsp15:mock_nsp15) %&gt;% 
  gather(virus, freq, MHVS_nsp15:mock_nsp15)

endoU_sub5S_freq_plot &lt;- endoU_subtractive_5S %&gt;% 
  ggplot(aes(x = dinuc , y = freq, fill = time)) + 
  scale_fill_brewer(palette = &#39;Set1&#39;) + 
  scale_x_discrete(limits=c(&quot;AA&quot;,&quot;GA&quot;, &quot;CA&quot;, &quot;UA&quot;,&quot;AC&quot;, &quot;CC&quot;, &quot;GC&quot;, &quot;UC&quot;, &quot;AG&quot;, &quot;CG&quot;, &quot;GG&quot;,     &quot;UG&quot;, &quot;AU&quot;, &quot;CU&quot;, &quot;GU&quot;, &quot;UU&quot;)) + 
  geom_bar(aes(fill = time), stat = &quot;identity&quot;, position = &#39;dodge&#39;) + 
  theme_cowplot() + 
  scale_y_continuous(limits = c(0.0, 0.06)) +
  facet_grid(cell ~ virus) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle(&quot;5S nsp15 subtractive dinucleotide frequency plots &quot;) 
endoU_sub5S_freq_plot</code></pre>
<p><img src="rRNA_endoU_analysis_files/figure-html/unnamed-chunk-10-3.png" width="672" /></p>
<div id="discussion-of-nsp15-subtractive-analysis" class="section level3">
<h3>Discussion of nsp15 subtractive analysis:</h3>
<p>For each rRNA, there were no obvious differences in dincuelotide cleavage preferecnce between the mock and infected samples. Whereas the RNase L subtractive analysis exposed some differences in dincleotide cleavage frequency between the mock and infected samples, which suggests that there is no nsp15 dependent cleavage preference in rRNA or that the cleavage freuqncy is less robust than RNase L and cannot be distinguished from the background.</p>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
