---
title: "Analysis of MHV RNA to Detect EndoU Cleavage Sites"
author: "Rachel Ancar"
date: "04/24/2018"
output:
  html_document:
    code_folding: hide
    toc: yes
    toc_float: yes
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
knitr::opts_chunk$set(message = FALSE)
```


```{r, message = F, warning = F}

library(tidyverse)
library(readr)
library(dplyr)
library(ggplot2)
library(valr)
library(stringr)
library(cowplot)
library(RColorBrewer)
library(purrr)
library(shiny)
library(DT)

```

2´,3´-cyclic phosphate RNA-sequencing libraries were prepared from mouse macrophages: B6 (WT), IFNAR KO, RNaseL KO. These cells were either mock infected or infected with WT MHV virus (from Volker or Susan), MHV mutated to inactivate ns2 (phosphodiesterase) activity, or MHV mutated to inactivate nsp15 (endoribonuclease) activity for 9 and 12 hours. 

```{r, message = F, warning = F}

sample_info_table <- read_tsv("sample_info.txt", col_names = c("cell type", "virus type", "time"))
datatable(sample_info_table)

```

These libraries were processed to generate bedgraph files containing sites of 3´-cleavage and the associated dinucleotide in the MHV (+) strand RNA (mRNA) and (-) strand RNA (genomic). 

##Coverage plots for all cell types by type of viral infection

These plots show the normalized counts of captured 2´,3´-cyclic phosphates captured (% of total cDNA reads for each library) per nucleotide position in the the MHV (+) strand reads. The MHV (-) strand reads are displayed using un-normalized counts because there were very few detected cleavage events and those occured at very low frequency. 

```{r, message = F, warning = F}

#MHV (+) strand coverage plots 

data_dir_neg = "~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/viral_bg/neg"
data_files_neg = list.files(data_dir_neg, full.names = T)

read_file <- function(x) {
  df <- readr::read_tsv(x, col_names = c("chrom", "start", "end", "count", "normalized_count", "dinuc"))
  df$name <- basename(x)
  df
}

neg_table <- purrr::map_df(data_files_neg, read_file) %>%
  mutate(name = str_replace(name, ".umi.dinuc.neg.bg", "")) %>%
  separate(name, into = c('cell', 'virus', 'time'), sep = '_')

neg_plot <- ggplot(neg_table, aes(x = start, y = normalized_count)) +
    geom_line(aes(color = time)) +
    scale_colour_brewer(palette="Set1") + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) +
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    labs(x="Position", y="Normalized counts (counts/total reads)") +
    ggtitle("MHV (+) strand coverage plots")
neg_plot

#MHV (-) strand coverage plots

data_dir_pos = "~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/viral_bg/pos"
data_files_pos = list.files(data_dir_pos, full.names = T)

pos_table <- purrr::map_df(data_files_pos, read_file) %>%
  mutate(name = str_replace(name, ".umi.dinuc.pos.bg", "")) %>%
  separate(name, into = c('cell', 'virus', 'time'), sep = '_')

pos_plot <- ggplot(pos_table, aes(x = start, y = count)) +
    geom_bar(aes(fill = time), stat = "identity", position = 'dodge') +
    scale_fill_brewer(palette = 'Set1') + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
    labs(x="Position", y="Counts") +
    ggtitle("MHV (-) strand coverage plots")
pos_plot

```

###Discussion of coverage analysis: 

1. We do not observe significant capture of MHV RNA from any of the mock infected samples
2. In general, we see more MHV "cleavage" (aka 2´,3´-cyclic phosphate capture) at 12 hpi vs. 9 hpi with WT virus, which may reflect greater viral RNA abundance at the later time point. In the ns2 mutant, which will result in increased RNaseL activation, we see even more robust cleavage, especially at 12 hours.
3. There is a significant decrease in signal in the viral samples with mutant nsp15, which may reflect decreased viral RNA abundance, adiditonally, we also see a different temporal pattern with the majority of cleavage at 9 vs. 12 hpi which may reflect earlier detection and cleavage of MHV RNA in the absence of nsp15. 
4. The IFNAR KO and RNaseL KO samples are similar, but different. This may reflect direct vs. infdirect effects of dsRNA sensing pathways and effectors on MHV detection.
5. In the IFNAR KO and RNaseL KO samples, there are lower amounts of MHV cleavage events detected, which could be a result of RNaseL activation. A measure of total viral RNA in these samples could provde some clarity. Additionally, the RNA clevaage events are more robust at 9 hpi than observed in the WT B6 samples. 
6. Although low abundance, the key comparison between WT MHV and mutant MHV across all cell types suggest that there may be some sites of specific nsp15 cleavage. 

##RNase L substractive coverage plots of MHV (+) strand reads for all cell types by type of viral infection

The normalized counts detected in RNase L KO cells for each type of viral infection were subtracted from the normalized counts detected in WT B6 and IFNAR KO cells. This substractive analysis will emphasizeRNase L dependent cleavage events by reporting signals that occur only in the presence of RNase L. 

```{r, message = F, warning = F}

#RNase L substractive analysis to remove signal that occurs in the abscence of RNAse L by cell type 

subtractive_neg <- neg_table %>% 
  dplyr::select(-count, -dinuc) %>% 
  spread(cell, normalized_count) 
subtractive_neg[is.na(subtractive_neg)] <- 0

subtractive_neg <- subtractive_neg %>% 
  mutate(B6_RNaseLdep = B6 - RNaseL) %>% 
  mutate(IFNAR_RNaseLdep = IFNAR - RNaseL) %>% 
  dplyr::select(start:end, virus:time, B6_RNaseLdep:IFNAR_RNaseLdep) %>% 
  gather(cell, normalized_count, B6_RNaseLdep:IFNAR_RNaseLdep)
                                      
#Plots of RNase L dependent sites 
subneg_plot <- ggplot(subtractive_neg, aes(x = start, y = normalized_count)) +
    geom_line(aes(color = time)) + 
    scale_color_brewer(palette = 'Set1') + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
    labs(x="Position", y="Normalized counts (counts/total reads)") +
    ylim(0, 0.015) + 
    ggtitle("RNase L dependent-cleavage sites analysis")
subneg_plot

```

##EndoU mutant (nsp15) substractive coverage analysis of MHV (+) strand for all cell types by type of viral infection

This is very similar to the analysis above, expect it will substract signal occuring in the abscence of nsp15 actviity, which will emphasize sites specific to nsp15 cleavage actviity. 

```{r, message = F, warning = F}

# nsp15 substractive analysis to remove signal that occurs in the abscence of nsp15 activity by viral infection 

subtractive_nsp15 <- neg_table %>% 
  dplyr::select(-count, -dinuc) %>% 
  spread(virus, normalized_count)
subtractive_nsp15[is.na(subtractive_nsp15)] <- 0

subtractive_nsp15 <- subtractive_nsp15 %>% 
  mutate(MHVS_nsp15dep = MHVS - nsp15) %>% 
  mutate(MHVV_nsp15dep = MHVV - nsp15) %>% 
  mutate(ns2_nsp15dep = ns2 - nsp15) %>% 
  dplyr::select(start:end, cell:time, MHVS_nsp15dep:ns2_nsp15dep) %>% 
  gather(virus, normalized_count, MHVS_nsp15dep:ns2_nsp15dep)

#Plots of nsp15 dependent sites 
                         
subnsp15_plot <- ggplot(subtractive_nsp15, aes(x = start, y = normalized_count, fill = time)) +
    geom_line(aes(color = time)) + 
    scale_colour_brewer(palette = 'Set1') + 
    theme_cowplot() +
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
    labs(x="Position", y="Normalized counts (counts/total reads)") +
    ylim(0, 0.015) + 
    scale_x_continuous(breaks=seq(0, 30000, 5000)) +
    ggtitle("Nsp15 dependent-cleavage sites analysis")
subnsp15_plot

```

###Discussion of substractive RNase L and Nsp15 analysis: 

We have captured signal specific to RNase L and Nsp15 cleavage of MHV + strand RNA. This is evidenced by the signal the RNase L or Nsp15-dependent signal that remains in nsp15-mutant RNase L WT cells and RNase L KO cells with WT nsp15. 
We can compare the sites of RNase L cleavage to data previously collected by Dave Barton, but there has been no previously identified sites of cleavge in MHV RNA by nsp15. 

Looking specifically at the nsp15-subtractive analysis, the robust signal in all of the B6 cells is nsp15-dpendent, but not RNaseL independent, because RNaseL could also be responsible for this signal. Looking in the RNaseL KO samples give us the best understanding of nsp15 cleavage of MHV RNA without the confounding effect of RNaseL cleavage. Focusing on this cell type, we 
see the nsp15-dependent and RNaseL-independent sites in MHV RNA at 12 and 9 hours after infection. There are some clear sites of cleavage that rise signal above the background noise. 

Furthermore, the nsp15-dependent sites in the cells infected with ns2-mutant MHV have have increased amounts of cleavage, slightly different patterns of cleavage, and cleavage that occurs early as compared to the WT virus. This suggests that there may be some time of interaction between ns2 and nsp15. 

##Examining specific sites of cleavage in MHV (+) strand RNA

By visual inspection, the nsp15 substractive analysis data suggests some potential regions of interest in the MHV genome that may be cleaved by endoU. Some specific sites are visualized below. 

Site18645: Previously identified by Dave Barton as potentially a site of interest. Dinucleotide is "GC" at 18645 (GA in reference, but GC in viral RNA). There is also a site of clevage with smaller signal at 18648, which is "UU" 

Site 26080 and 26087: 26080 = UC, 26087 = CU

Site25404: 25404 = UC

Site 26147 and 26148: 26147 = UC, 26147 = UU

```{r, message = F, warning = F}

#Site18645-Previously identified in unpublished data from Dave Barton as potentially a site of interest. Dinucleotide is "GC" at 18645 (GA in reference, but GC in viral RNA). There is also a site of clevage with smaller signal at 18648, which is "UU" 

neg_plot_18645 <- filter(neg_table, time != "012") %>% filter(time != "09") %>%
  ggplot(aes(x = start, y = normalized_count, fill = time)) +
    geom_bar(stat = "identity", position = 'dodge') +
    scale_fill_brewer(palette="Set1") + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
    labs(x="Position", y="Normalized counts (counts/total reads)") +
    xlim(18640, 18660) +
    ggtitle("MHV (+) strand coverage plot: sites 18645 and 18648")
neg_plot_18645

#Site 26080 and 26087- identified by visual inspection of the data, 26080 = UC, 26087 = CU

neg_plot_26080 <- filter(neg_table, time != "012") %>% filter(time != "09") %>%
    ggplot(aes(x = start, y = normalized_count, fill = time)) +
    geom_bar(stat = "identity", position = 'dodge') +
    scale_fill_brewer(palette="Set1") + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
    labs(x="Position", y="Normalized counts (counts/total reads)") +
    xlim(26075, 26090) +
    ggtitle("MHV (+) strand coverage plot: sites 26080 and 26087")
neg_plot_26080 
  
#Site25404 - identified by visual inspection of the data, 25404 = UC

neg_plot_25404 <- filter(neg_table, time != "012") %>% filter(time != "09") %>%
    ggplot(aes(x = start, y = normalized_count, fill = time)) +
    geom_bar(stat = "identity", position = 'dodge') +
    scale_fill_brewer(palette="Set1") + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
    labs(x="Position", y="Normalized counts (counts/total reads)") +
    xlim(25390, 25410) +
    ggtitle("MHV (+) strand coverage plot: site 25404")
neg_plot_25404

#Site 26147 and 26148 - identified by visual inspection of the data, 26147 = UC, 26147 = UU

neg_plot_26148 <- filter(neg_table, time != "012") %>% filter(time != "09") %>%
    ggplot(aes(x = start, y = normalized_count, fill = time)) +
    geom_bar(stat = "identity", position = 'dodge') +
    scale_fill_brewer(palette="Set1") + 
    theme_cowplot() + 
    facet_grid(virus ~ cell) + 
    theme(axis.text.x = element_text(angle = 90, hjust = 1)) + 
    labs(x="Position", y="Normalized counts (counts/total reads)") +
    xlim(26142, 26150) +
    ggtitle("MHV (+) strand coverage plot: sites 26147 and 26148")
neg_plot_26148

```

##Identifying "significant" sites of possible nsp15 cleavage

Visual inspection of the data indicates that there are some nsp15 cleavage sites in the MHV (+) RNA. However, it is difficult to ascertain which sites are "real" because of the level of background cleavage and the low signal of the cleavage events that have been detected. This analysis attempts to calculate which potential cleavage sites have the greatest fold change in cleavage singal captured between samples infected with WT virus and samples infected with nsp15 mutant virus. Cleavage sites where signal captured during WT infection decreases the most during infection with nsp15 mutant virus may be "real" sites of MHV endoU cleavage acitivity. 

Calculating "fold change" in cleavage between nsp15 virus infections and WT virus across all cell types

```{r, message = F, warning = F}

change_nsp15 <- neg_table %>% 
  dplyr::select(-count, -dinuc) %>% 
  spread(virus, normalized_count)

change_nsp15[is.na(change_nsp15)] <- 0

change_nsp15 <- change_nsp15 %>% 
  mutate(MHVS_nsp15change = nsp15/MHVS) %>% 
  mutate(MHVV_nsp15change = nsp15/MHVV) %>% 
  mutate(ns2_nsp15change = nsp15/ns2) %>% 
  gather(viral_comp, fold_change, MHVS_nsp15change:ns2_nsp15change) %>%
  filter(fold_change > 0 & fold_change <= 1) %>%
  arrange(desc(fold_change))

```

####Ordering fold change values by start position and top 50 decreased sites by fold change by each cell type and time

```{r, message = F, warning = F}

new_changensp15 <- change_nsp15 %>% 
  group_by(cell, time, viral_comp) %>%
  arrange(desc(fold_change)) %>% 
  group_by(chrom, start, end, cell, time, viral_comp) %>%
  arrange(desc(fold_change)) %>%
  ungroup() %>%
  arrange(start) %>%
  group_by(cell, time) %>%
  arrange(fold_change) %>%
  do(head(., n = 50)) %>%
  mutate(log2foldchange = -log2(fold_change))
datatable(new_changensp15)

```

####Plotting > 4 foldchange sites 

```{r, message = F, warning = F}

foldchange_graph <- filter(new_changensp15, log2foldchange >= 4) %>%
  ggplot(aes(x = start, y = log2foldchange, color = cell)) +
  geom_bar(stat = "identity", position = 'dodge') + 
  scale_fill_brewer(palette="Set1") +
  facet_grid(viral_comp ~ cell) +
  theme_cowplot() +
  labs(x="Position", y="Log2foldchange") +
  theme(axis.text.x = element_text(angle = 90, hjust = 1))
foldchange_graph

#Graph of log2foldchange for site 21307
site21307_foldchange_graph <- filter(new_changensp15, start == 21307) %>%
  ggplot(aes(x = viral_comp, y = log2foldchange, fill = cell)) +
  geom_bar(stat = "identity", position = 'dodge', width = 0.4) + 
  scale_fill_brewer(palette="Set1") +
  theme_cowplot() + 
  labs(x="viral_comp", y="Log2foldchange") +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  ggtitle("log2foldchange for site 21307")
site21307_foldchange_graph

```

####Table: B6 cells at 12 hrs, fold change and dinuc for top 50 sites with the greatest decrease in signal 

```{r, message = F, warning = F}

B6_12 <- new_changensp15 %>% 
  filter(cell == "B6" & time == "12")
dinuc_positions <- neg_table %>% 
  dplyr::select(chrom, start, end, dinuc)
B6_12_dinuc <- left_join(B6_12, dinuc_positions) %>% 
  unique() %>% 
  dplyr::select(chrom, start, end, log2foldchange, dinuc)
datatable(B6_12_dinuc)

```

#### Table: RNaseL KO cells at 12 hrs, fold change and dinuc

```{r, message = F, warning = F}

RNaseL_12 <- new_changensp15 %>% 
  filter(cell == "RNaseL" & time == "12")
dinuc_positions <- neg_table %>% 
  dplyr::select(chrom, start, end, dinuc)
RNaseL_12_dinuc <- left_join(RNaseL_12, dinuc_positions) %>% 
  unique() %>%
  dplyr::select(chrom, start, end, log2foldchange, dinuc)
datatable(RNaseL_12_dinuc)

```

###Discussion of analysis to identify significant nsp15 cleavage sites in MHV RNA

Visual inspection of the data can identify potential sites of cleavage in MHV RNA. However, many of these sites have significant cleavage in the presence of WT nsp15, but in many cases some signal remains with mutant nsp15. It is difficult to determine whether this remaining signal is background non-specific signal or indicates that another nucleases is contributing to our detected signal. Furthermore, the pattern and abundance of cleavage at these potnetial sites varies by cell types and to some extent between the 2 WT viruses. 

The fold change analysis is a useful way to screen potential cleavage sites of interest.

Looking at the sites that decrease at least 4-fold, there seems to be more overlap within cell type than between. However, there are many sites where decreased cleavage was only detected in one cell type or type of viral infection. The table containing the top 50 sites for each cell type and time can be used to investigate specific sites of interest. Additionally, adding dinucleotide data to the top 50 sites per cell and time provides another method to screen in and out potential endoU cleavage sites. For example, the site 18645 does not decrease greater than 4-fold in any of the conditions because although there is robust signal in the prescence of WT nsp15, signal remains at this sight in the nsp15 mutant. So, the question is whether we are more interested in sites with the most robust signal, but less of a decrease in capture rate in the nsp15 mutant or sites that had lower intial signal, but also a greater decrease in signal in the nsp15 mutant. 

The sites with the greatest decrease in cleavage in B6 cells at 12 hpi are enriched in the dinucleotides "UC", "UU", "CC", "CU". In this cell line, RNaseL is present so some of these sites are presumably RNaseL dependent. Additionally, because the signal across all sites is lower in the cells infected with nsp15 virus vs. WT MHV, the background level of signal decrease is around 3-fold. So, the plot only includes data for sites that decrease 4-fold or more. 

In the RNaseL KO cells at 12 hpi, this bias is lost. The sites with the greatest decrease in signal are at the dinucleotides "GA", "UU", "GU", "AG", "CG". Although, the data in the RNaseL KO cells has the opposite problem described for the B6 cells with WT and mutant virus. There is overall decreased signal in the RNaseL KO cells, so this should be taken into consideration when interpreting the fold change analysis. 

In additiona to screening sites of interest, this analysis also helps to identify cleavage sites. The top changing site in B6 cells is also present in the RNaseL cells at the same position, as illustrated in the last graph above. This site is at a "UC" dinucleotide and may be a good candidtae cleavage site to confirm with other methods. This type of analysis can be expanded on to focus in on specific parameters, such as sites only present across cell type, across viral type, in certain regions of the viral RNA, etc...

##Calculating dinucelotide frequencies for all cell types by infection status 
Performing a dinucleotide frequency analysis will help us to confirm RNaseL signatures as a positive control and investigate a potential sequence preference for nsp15 cleavage.

```{r, message = F, warning = F}

# Calculating freuqency of each dinucleotide for the + strand MHV reads 

dinuc <- c("AA","GA", "CA", "UA","AC", "CC", "GC", "UC", "AG", "CG", "GG", "UG", "AU", "CU", "GU", "UU")
df_dinuc <- data_frame(dinuc)

frequencies_neg <- function(x) {
  df <- readr::read_tsv(x, col_names = c("chrom", "start", "end", "count",      "normalized_count", "dinuc"))
  df <- aggregate(cbind(count) ~ dinuc, data = df, sum)
  df <- left_join(df_dinuc, df) %>% 
    replace_na(list(count = 0)) %>% 
    mutate(freq = count/sum(count)) 
  df$name <- basename(x)
  df %>% mutate(name = str_replace(name, ".umi.dinuc.neg.bg", "")) %>%
  separate(name, into = c('cell', 'virus', 'time'))
}

neg_freq <- purrr::map_df(data_files_neg, frequencies_neg)

```

##Plotting dinucleotide frequencies for all cell types by type of viral infection

These plots show the freuqency of cleavege at 3´-dinucleotides for the MHV (+) sense RNA from highest to lowest frequency. I removed the mock infected sample data from these graphs. 

```{r, message = F, warning = F}

# Dincleotide freuqnecy plots 

neg_freq_plot <- filter(neg_freq, virus != "mock") %>%
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = 'Set1') + 
  scale_x_discrete(limits=c("AA","GA", "CA", "UA","AC", "CC", "GC", "UC", "AG", "CG", "GG",   "UG", "AU", "CU", "GU", "UU")) + 
  geom_bar(aes(fill = time), stat = "identity", position = 'dodge') + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle("MHV (+) strand dinucleotide frequency plots")
neg_freq_plot

```

## RNase L substractive dinucleotide analysis of for all cell types by type of viral infection

This is similar to the previous coverage substractive analysis. These plots display dinucleotde cleavage frequencies dependent on RNase L by removing the frequencies detected in the abscence of RNase L. 

```{r, message = F, warning = F}

# Subtractive RNaseL dinucleotide analysis

subtractive_neg <- neg_freq %>% 
  dplyr::select(dinuc, freq:time) %>% 
  spread(cell, freq) %>% 
  mutate(B6_RNaseLdep = B6 - RNaseL) %>% 
  mutate(IFNAR_RNaseLdep = IFNAR - RNaseL) %>% 
  dplyr::select(dinuc:time, B6_RNaseLdep:IFNAR_RNaseLdep) %>% 
  gather(cell, freq, B6_RNaseLdep:IFNAR_RNaseLdep)

# Plots 

subneg_freq_plot <- subtractive_neg %>% 
  filter(virus != "mock") %>%
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = 'Set1') + 
  scale_x_discrete(limits=c("AA","GA", "CA", "UA","AC", "CC", "GC", "UC", "AG", "CG", "GG",     "UG", "AU", "CU", "GU", "UU")) + 
  geom_bar(aes(fill = time), stat = "identity", position = 'dodge') + 
  theme_cowplot() +
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle("MHV (+) strand RNase L subtractive dinucleotide frequency analysis")
  #scale_y_continuous(limits = c(0.00, 0.10))
subneg_freq_plot


```

## Nsp15 substractive dinucleotide analysis of for all cell types by type of viral infection

These plots indentify some dinucleotide cleavage events that might be dependent on nsp15 activity and indenpendent of RNase L activity. 

```{r, message = F, warning = F}

#Subtractive nsp15 dinucleotide analysis

subdinuc_neg_nsp15 <- neg_freq %>% 
  dplyr::select(dinuc, freq:time) %>% 
  spread(virus, freq) %>%
  mutate(MHVS_nsp15dep = MHVS - nsp15) %>% 
  mutate(MHVV_nsp15dep = MHVV - nsp15) %>% 
  mutate(ns2_nsp15dep = ns2 - nsp15) %>% 
  dplyr::select(dinuc:time, MHVS_nsp15dep:ns2_nsp15dep) %>% 
  gather(virus, freq, MHVS_nsp15dep:ns2_nsp15dep)

# Plots

subnegdinuc_freq_plot <- subdinuc_neg_nsp15 %>% 
  filter(virus != "mock") %>% 
  filter(time != "012") %>% 
  filter(time != "09") %>% 
  ggplot(aes(x = dinuc , y = freq)) + 
  scale_fill_brewer(palette = 'Set1') + 
  scale_x_discrete(limits=c("AA","GA", "CA", "UA","AC", "CC", "GC", "UC", "AG", "CG", "GG",   "UG", "AU", "CU", "GU", "UU")) + 
  geom_bar(aes(fill = time), stat = "identity", position = 'dodge') + 
  theme_cowplot() +
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  scale_y_continuous(limits = c(0.00, 0.10)) +
  ggtitle("MHV (+) strand nsp15 subtractive dinucleotide frequency analysis")
subnegdinuc_freq_plot

```

### Discussion of dinucleotide analysis

The most prominent cleavages are consistent with known RNaseL dinucleotide preferences, UA and UU. You can see this predominantly in the ns2 mutant virus infection greater than in the WT MHV and greater still in the samples with mutant ns15, which is a pattern we have also observed in the coverage analysis. 

The UA cleavage frequency is significantly decreased in the IFNAR and RNaseL samples, Furthermore, we see a prominent signal at UC, which could represent a endoU preference, but does not much in the nsp15 mutant. Furthermore, we note general lack of cleavage at G, and a similar dearth of cleavage at A residues, a pattern that does seem to shift in the RNaseL KO samples. 

The RNaseL substractive analysis reconfirms that preference for RNaseL cleavage at UA and UU and also suggests that UG may be may also be partially RNaseL dependent. 

The nsp15 substractive analysis in the B6 cells reaveals a clear pattern where dinucleotides where the 3´-position is C and U are favored. However, since RNase L is still active in these cells, we know that RNase L cleavage preferences are contributing to some of this observed pattern. In the IFNAR and RNase L KO cells, this pattern is mostly lost. 

## Single nucleotide analysis of nucleotide enrichement at the 3´-clevaage site

Previous reports have identified that endoU prefers to cleave 3´of U and C with a greater preference for U >> C. This analysis calculates the frequency of capture of each dinucleotide per cell type, virus type, and time point. 

```{r, message = F, warning = F}

#Calculating nucleotide freuqnecies for + strand MHV RNA

data_dir_nuc = "~/Dropbox (Hesselberth Lab)/Rachel_data/EndoU_project/viral_bg/nucleotide_analysis"
data_files_nuc = list.files(data_dir_nuc, full.names = T)

nuc <- c("A","G", "C", "U")
df_nuc <- data_frame(nuc)

frequencies_neg <- function(x) {
  df <- readr::read_tsv(x, col_names = c("chrom", "start", "end", "count", "normalized_count", "nuc"))
  df <- aggregate(cbind(count) ~ nuc, data = df, sum)
  df <- left_join(df_nuc, df) %>% 
    replace_na(list(count = 0)) %>% 
    mutate(freq = count/sum(count)) 
  df$name <- basename(x)
  df %>% mutate(name = str_replace(name, ".umi.nuc.neg.bg", "")) %>%
  separate(name, into = c('cell', 'virus', 'time'))
}

neg_nuc_freq <- purrr::map_df(data_files_nuc, frequencies_neg)

# Plots of nucleotide frequency across all cell types and viral infections

neg_nuc_freq_plot <- filter(neg_nuc_freq, virus != "mock") %>%
  ggplot(aes(x = nuc , y = freq)) + 
  scale_fill_brewer(palette = 'Set1') + 
  scale_x_discrete(limits=c("A","G", "C", "U")) + 
  geom_bar(aes(fill = time), stat = "identity", position = 'dodge') + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
  ggtitle("Single nuceltoide frequency analysis")
neg_nuc_freq_plot

# Calculating nucleotide freuqnecies with nsp15 subtractive analysis 

subnuc_neg_nsp15 <- neg_nuc_freq %>% 
  dplyr::select(nuc, freq:time) %>% 
  spread(virus, freq) %>%
  mutate(MHVS_nsp15dep = MHVS - nsp15) %>% 
  mutate(MHVV_nsp15dep = MHVV - nsp15) %>% 
  mutate(ns2_nsp15dep = ns2 - nsp15) %>% 
  dplyr::select(nuc:time, MHVS_nsp15dep:ns2_nsp15dep) %>% 
  gather(virus, freq, MHVS_nsp15dep:ns2_nsp15dep)

# Plotting single nucleotide nsp15 substractive analysis 

subneg_nuc_nsp15_plot <- filter(subnuc_neg_nsp15, virus != "mock") %>% 
  filter(time != "012") %>% 
  filter(time != "09") %>% 
  ggplot(aes(x = nuc , y = freq)) + 
  scale_fill_brewer(palette = 'Set1') + 
  scale_x_discrete(limits=c("A","G", "C", "U")) + 
  geom_bar(aes(fill = time), stat = "identity", position = 'dodge') + 
  theme_cowplot() + 
  facet_grid(virus ~ cell) + 
  theme(axis.text.x = element_text(angle = 90, hjust = 1)) +
   ggtitle("nsp15 substractive single nuceltoide frequency analysis")
subneg_nuc_nsp15_plot

```

### Discussion of single nucleotide analysis: 

This analysis reveals a bias for U and C across all cell types and viral infection conditions. G is counted at the lowest frequency consistently. However, the U bias does not change significantly in the RNase L KO or the nsp15 mutant virus infection. This suggests that there is increased cleavage at U's indepdnent of RNase L or nsp15 or may be a result of overpresentation of U's in the MHV (+) RNA as compared to the other nucelotides. The capture of C's does seem to increase in RNase L KO cells and decrease minimally in the nsp15 mutant infection. 

The nsp15 substractive analysis reveals that in B6 cells, cleavge at C and U is nsp15 dependent. However, this clear signal decreases sigficantly in RNase L KO cells, but there does seem to be some residual U and C signal in the Volker WT virus and Ns2 mutant virus, with the most signal at C residues. It may be useful to perform a motif analysis to determine if a broader sequence context directs nsp15 cleavage. 

